// @siroop/kvasir/source/70-templates/cart/cart.js
// const selectors = {
// 	sidebar: '.js-t-cart__aside',
// 	footer: '.js-page-footer',
// };
//
// class Cart {
// 	constructor($el) {
// 		this.el = $el[0];
// 		this.window = window;
// 		this.document = document;
// 		this.body = this.document.body;
// 		this.sidebar = this.el.querySelectorAll('.js-t-cart__aside')[0];
// 		this.footer = this.body.querySelectorAll('.js-page-footer')[0];
// 		this.window.addEventListener('scroll', () => this.onScrollMove());
// 	}
//
// 	// on scroll move the sidebar
// 	onScrollMove(){
// 		console.log('is it even fired');
// 		this.sidebarHeight = this.sidebar.clientHeight;
// 		this.footerOffsetTop = this.footer.getBoundingClientRect().top;
// 		this.sidebarOffset = this.sidebar.getBoundingClientRect();
//
// 		if(this.window.scrollTo(0,0) > this.sidebarOffset.top) {
// 			this.sidebar.style.position = 'fixed';
// 			this.sidebar.style.top = '0';
// 		} else {
// 			this.sidebar.style.position = 'relative';
// 		}
// 		if(this.window.scrollTo(0,0) + this.sidebarHeight > this.footerOffsetTop) {
// 			const position = -(this.window.scrollTo(0,0) + this.sidebarHeight - this.footerOffsetTop);
// 			this.sidebar.style.top = position;
// 		} else {
// 			this.sidebar.style.top = '0';
// 		}
// 	}
// }
//
// export default Cart;
//
// $(document).ready(() => {
// 	const $window = $(window);
// 	const $sidebar = $('.js-o-cart-aside');
// 	const $sidebarHeight = $sidebar.innerHeight();
// 	const $footerOffsetTop = $('.js-page-footer').offset().top;
// 	const $sidebarOffset = $sidebar.offset();
// 	console.log($sidebarHeight);
//
// 	$window.scroll(() => {
// 		if ($window.scrollTop() > $sidebarOffset.top) {
// 			$sidebar.css({ 'position': 'fixed' });
// 			$sidebar.css({ 'top': '0' });
// 			$sidebar.css({ 'width': 'inherit' });
// 		} else {
// 			$sidebar.css({ 'position': 'relative' });
// 			$sidebar.css({ 'width': '100%' });
// 		}
// 		if ($window.scrollTop() + $sidebarHeight > $footerOffsetTop) {
// 			$sidebar.css({ top: -($window.scrollTop() + $sidebarHeight - $footerOffsetTop) });
// 		} else {
// 			$sidebar.css({ top: '0' });
// 		}
// 	});
// });
