// @siroop/kvasir/source/70-templates/cart/cart.config.js
const cartAside = require('../../60-components/66-organisms/cart-aside/cart-aside.config');
const cartTotal = require('../../60-components/66-organisms/cart-total/cart-total.config');
const cartProductList = require('../../60-components/66-organisms/cart-product-list/cart-product-list.config');
const cartAuth = require('../../60-components/64-molecules/cart-auth/cart-auth.config');

module.exports = {
	title: 'Cart',
	name: 'cart',
	context: {
		data: {
			title: 'Warenkorb',
			subTitle: 'Artikel im Warenkorb werden nicht reserviert',
			products: cartProductList.context.data,
			cartAside: cartAside.context.data,
			cartTotal: cartTotal.context.data,
			cartAuth: cartAuth.context.data,
			voucher: {
				title: 'Gutschein',
				description: 'Jack Wolfskin Jacke Colorado Flex - Midnight Blue',
				labelValidUntil: 'Gültig bis',
				discount: '10%',
				tooltip: {
					text: 'Rundungsdifferenzen können nicht ausgeschlossen werden, der totale Rabattbetrag kann daher um wenige Rappen variieren. Rundungsdifferenzen können nicht ausgeschlossen werden, der totale Rabattbetrag kann daher um wenige Rappen variieren.',
					modifier: 'medium',
				},
				labelExpired: 'Voucher ist nicht mehr gültig',
				buttonRemove: {
					text: 'Gutschein entfernen',
				},
			},
		},
	},
};
