// @siroop/kvasir/source/00-data/search/index.js
module.exports = {
	items: [
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7',
				price: '948.75',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-0715737',
				image_url: 'https://cdn.siroop.ch/media/images/sized/2b/ea/68/2bea68f778325072fbe94c8561938674.50x50.jpg',
				group_id: 10638,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7 Plus',
				price: '822.65',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-plus-0940654',
				image_url: 'https://cdn.siroop.ch/media/images/sized/16/bc/d6/16bcd6d51c336c8e0e15d551b66fb85c.50x50.jpg',
				group_id: 10646,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7 Plus very long name to demonstrate a truncated line very long name to demonstrate a truncated line',
				price: '1028.00',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-plus-0747790',
				image_url: 'https://cdn.siroop.ch/media/images/sized/4c/7f/f9/4c7ff9ffe8912d04a245f7b90068e9c6.50x50.jpg',
				group_id: 10646,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7',
				price: '701.40',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-0715739',
				image_url: 'https://cdn.siroop.ch/media/images/sized/79/ec/f6/79ecf692faea8f66747d808c114ad5fd.50x50.jpg',
				group_id: 10638,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7 Plus',
				price: '720.95',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-plus-128-gb-gold-0715740',
				image_url: 'https://cdn.siroop.ch/media/images/sized/55/78/03/5578036c3ae436727efd0625802c2240.50x50.jpg',
				group_id: 10646,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7',
				price: '789.00',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-0852006',
				image_url: 'https://cdn.siroop.ch/media/images/sized/72/d3/3a/72d33aea572cc6cf1aa4f1f225b0f1f6.50x50.jpg',
				group_id: 10638,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7',
				price: '699.00',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-0715738',
				image_url: 'https://cdn.siroop.ch/media/images/sized/0b/86/13/0b8613824b420ea50c425dd1fe83a08b.50x50.jpg',
				group_id: 10638,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7 Plus',
				price: '899.00',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-plus-0747901',
				image_url: 'https://cdn.siroop.ch/media/images/sized/01/fe/ff/01fefff56b9a2d83a11ed1988c280595.50x50.jpg',
				group_id: 10646,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7',
				price: '698.00',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-0715736',
				image_url: 'https://cdn.siroop.ch/media/images/sized/9f/6a/6c/9f6a6c8149f0bc90e9c1ff923b18feaf.50x50.jpg',
				group_id: 10638,
			},
		},
		{
			type: 'suggest-product',
			data: {
				name: 'Apple iPhone 7 32 GB Black',
				price: '700.65',
				url: '/computer-elektronik/smartphone-handy/smartphone/apple-iphone-7-32-gb-black-0728148',
				image_url: 'https://cdn.siroop.ch/media/images/sized/48/fa/63/48fa63018e7fd343dd9353d656d331d5.50x50.jpg',
				group_id: 'na',
			},
		},
		{
			type: 'suggest-category',
			data: {
				name: 'Smartphone',
				url: '/computer-elektronik/smartphone-handy/smartphone?q=iphone 7',
				product_count: 48,
				queryString: 'iphone 7',
			},
		},
		{
			type: 'suggest-category',
			data: {
				name: 'Küchenhelfer',
				url: '/wohnen-haushalt/kochen-backen/kuechenhelfer?q=iphone 7',
				product_count: 1,
				queryString: 'iphone 7',
			},
		},
		{
			type: 'suggest-category',
			data: {
				name: 'PC Kabel & Adapter',
				url: '/computer-elektronik/pc-peripherie/pc-kabel-adapter?q=iphone 7',
				product_count: 9,
				queryString: 'iphone 7',
			},
		},
	],
	summary: {
		text: 'Alle 887 Produkte anzeigen',
	},
};
