// @siroop/kvasir/source/70-templates/customer-payment/customer-payment.config.js
const paymentMethods = require('../../00-data/payment');
const navigationList = require('../../60-components/64-molecules/list-navigation/list-navigation.config');


module.exports = {
	title: 'Customer payment',
	name: 'customer-payment',
	context: {
		data: {
			payment: {
				items: [
					{
						title: 'Meine Visa',
						labelDelete: 'Löschen',
						method: paymentMethods.visa,
					},
					{
						title: 'Meine Mastercard',
						labelDelete: 'Löschen',
						method: paymentMethods.mastercard,
					},
					{
						title: 'Meine Visa',
						labelDelete: 'Löschen',
						method: paymentMethods.postfinance,
					},
					{
						title: 'Meine American Express',
						labelDelete: 'Löschen',
						method: paymentMethods.paypal,
					},
				],
			},
			title: 'Meine Zahlungsarten',
			breadcrumb: [
				{
					url: '/',
					name: 'Startseite',
				},
				{
					url: '/components/preview/t-customer',
					name: 'Mein Konto',
				},
				{
					name: 'Meinen Adressen',
				},
			],
			navigation: navigationList.context.data,
		},
	},
};
