// @siroop/kvasir/source/00-data/navigation/brands.js
const multiplyItems = (qtyItems, item) =>
	Array.from({ length: qtyItems }, () => item);

const brand = {
	url: '/components/preview/t-brand-page',
	image: {
		srcset: [
			{
				src: '/img/content/brand-logos/lego.png',
				width: '100w',
			},
			{
				src: '/img/content/brand-logos/lego.png',
				width: '200w',
			},
			{
				src: '/img/content/brand-logos/lego.png',
				width: '400w',
			},
			{
				src: '/img/content/brand-logos/lego.png',
				width: '1000w',
			},
			{
				src: '/img/content/brand-logos/lego.png',
				width: '2000w',
			},
		],
		srcsetString: '/img/content/brand-logos/lego.png 100w, /img/content/brand-logos/lego.png 200w, /img/content/brand-logos/lego.png 400w, /img/content/brand-logos/lego.png 1000w, /img/content/brand-logos/lego.png 2000w',
		src: '/img/content/brand-logos/lego.png',
		alt: 'A responsive image',
		sizes: '',
	},
};

const alphabet = [
	'a', 'b', 'c', 'd', 'e', 'f', 'g',
	'h', 'i', 'j', 'k', 'l', 'm', 'n',
	'o', 'p', 'q', 'r', 's', 't', 'u',
	'v', 'w', 'x', 'y', 'z',
];

const alphabetNav = () => (
	alphabet.map(item => ({
		url: `/components/preview/t-brand-page-overview#${item}`,
		text: item,
	}))
);

// In the future there will be more than one object (top brands and all brands)
const brandItems = [
	{
		type: 'top-brands',
		title: 'Top Marken',
		url: '#top-marken',
		items: multiplyItems(12, brand),
	},
	{
		type: 'all-brands',
		title: 'Alle Marken',
		url: '#top-marken',
		showAllBrandsUrl: '/components/preview/t-brand-page-overview',
		showAllBrandsLabel: 'Alle Brands anzeigen',
		items: alphabetNav(),
	},
];


const brands = {
	title: 'Marken',
	url: '#marken',
	isActive: true,
	items: brandItems,
};

module.exports = brands;
