// @siroop/kvasir/source/60-components/66-organisms/filter-panel/filter-panel.config.js
module.exports = {
	title: 'Filter Panel',
	name: 'filter-panel',
	context: {
		modifier: '',
		data: {
			back: 'Zurück',
			buttons: [
				{
					name: '156 Ergebnisse anzeigen',
					modifiers: ['filled'],
				},
				{
					name: 'Abbrechen',
					modifiers: ['borders', 'small'],
					classes: ['js-o-filter-panel__back'],
				},
			],
			triggerButton: {
				default: true,
				class: 'o-filter-panel__button',
				text: 'Open panel',
			},
			activeFilters: {
				resetFilter: 'Alle Filter zurücksetzen',
				items: [
					{
						text: 'Marke: <b>Blackberry</b>',
					},
					{
						text: 'Farbe: <b>Blau</b>',
					},
					{
						text: 'Farbe: <b>Blau</b>',
					},
					{
						text: 'Farbe: <b>Blau</b>',
					},
				],
			},
			name: 'Filter',
			items: [
				{
					name: 'Preis',
					active: true,
					buttons: [
						{
							name: 'Anwenden',
							modifiers: ['filled'],
						},
					],
					content: {
						name: 'Preis',
						component: {
							name: 'o-price-slider',
							data: {
								id: '1',
								currency: 'CHF',
								button: {
									text: 'Anwenden',
								},
								start: {
									min: 5,
									max: 880,
								},
								range: {
									min: 5,
									max: 880,
								},
								label: {
									from: 'Von',
									to: 'Bis',
								},
							},
						},
					},
				},
				{
					name: 'Marke',
					active: true,
					content: {
						name: 'Marke',
					},
				},
				{
					name: 'Farbe',
					active: true,
					content: {
						name: 'Farbe',
					},
				},
				{
					name: 'Marke',
					content: {
						name: 'Marke',
					},
				},
				{
					name: 'Farbe',
					content: {
						name: 'Farbe',
					},
				},
				{
					name: 'Marke',
					content: {
						name: 'Marke',
					},
				},
				{
					name: 'Farbe',
					content: {
						name: 'Farbe',
					},
				},
				{
					name: 'Marke',
					content: {
						name: 'Marke',
					},
				},
				{
					name: 'Farbe',
					content: {
						name: 'Farbe',
					},
				},
				{
					name: 'Marke',
					content: {
						name: 'Marke',
					},
				},
				{
					name: 'Farbe',
					content: {
						name: 'Farbe',
					},
				},
				{
					name: 'Marke',
					content: {
						name: 'Marke',
					},
				},
				{
					name: 'Farbe',
					content: {
						name: 'Farbe',
					},
				},
				{
					name: 'Marke',
					content: {
						name: 'Marke',
					},
				},
			],
		},
	},
};
