// @siroop/kvasir/source/00-data/brand/index.js
const brands = [
	{
		name: 'Apple',
		slogan: 'Think different',
		logo: '/img/content/merchants/apple/apple-logo.png',
		headerImage: {
			srcset: [
				{
					src: '/img/content/merchants/apple/apple-cover-small.jpg',
					width: '767w',
				},
				{
					src: '/img/content/merchants/apple/apple-cover.jpg',
					width: '1200w',
				},
			],
			srcsetString: '/img/content/merchants/apple/apple-cover-small.jpg 767w, /img/content/merchants/apple/apple-cover.jpg 1200w',
			src: '/img/content/merchants/apple/apple-cover.jpg',
			alt: 'Header Image',
			sizes: '',
		},
		brandUrl: {
			href: '#',
			text: 'Alle Apple Produkte',
		},
		image: '',
		series: [
			{
				name: 'Summit Series',
				slogan: 'The worlds finest computers',
				urlText: 'Mehr über Pro Series',
				serieUrl: {
					href: '#',
					text: 'Alle Apple Produkte',
				},
			},
			{
				name: 'Summit Series',
				slogan: 'The worlds finest computers',
				urlText: 'Mehr über Pro Series',
				serieUrl: {
					href: '#',
					text: 'Alle Apple Produkte',
				},
			},
		],
	},
	{
		name: 'Lego',
		slogan: 'Lego',
		logo: 'https://cms.siroop.ch/wp-full/642e9ba8527ed202f017b970e4d127bec901d1d1/lego-340x192.png',
		headerImage: {
			srcset: [
				{
					src: 'https://cms.siroop.ch/wp-full/642e9ba8527ed202f017b970e4d127bec901d1d1/lego-340x192.png',
					width: '767w',
				},
				{
					src: 'https://cms.siroop.ch/wp-full/642e9ba8527ed202f017b970e4d127bec901d1d1/lego-340x192.png',
					width: '1200w',
				},
			],
			srcsetString: 'https://cms.siroop.ch/wp-full/642e9ba8527ed202f017b970e4d127bec901d1d1/lego-340x192.png 767w,' +
			' https://cms.siroop.ch/wp-full/642e9ba8527ed202f017b970e4d127bec901d1d1/lego-340x192.png 1200w',
			src: 'https://cms.siroop.ch/wp-full/642e9ba8527ed202f017b970e4d127bec901d1d1/lego-340x192.png',
			alt: 'Header Image',
			sizes: '',
		},
		brandUrl: {
			href: '#',
			text: 'Alle Lego Produkte',
		},
		image: '',
		series: [
			{
				name: 'Duplo',
				slogan: 'Duplo slogan',
				urlText: 'Mehr über Duplo Series',
				serieUrl: {
					href: '#',
					text: 'Alle Duplo Produkte',
				},
			},
		],
	},
];

module.exports = brands;
