// @siroop/kvasir/source/70-templates/faq/faq.config.js
module.exports = {
	title: 'FAQ page',
	name: 'faq-page',
	context: {
		data: {
			title: 'Häufige Fragen und Kontakt',
			categories: [
				{
					category: 'siroop',
					url: '#siroop',
				},
				{
					category: 'Bestellung & Lieferung',
					url: '#bestellung-&-lieferung',
				},
				{
					category: 'Retoure',
					url: '#retoure',
				},
				{
					category: 'Garantie',
					url: '#garantie',
				},
				{
					category: 'Bezahlung & Gutschriften',
					url: '#bezahlung-&-gutschriften',
				},
				{
					category: 'Social Media Guidelines',
					url: '#social-media-guidelines',
				},
				{
					category: 'AGB',
					url: '#agb',
				},
				{
					category: 'Kontakt',
					url: '#kontakt',
				},
			],
			questions: [
				{
					question: 'Wer seid ihr?',
					answer: 'siroop ist ein Online-Marktplatz, ' +
                    'der für dich das Beste aus der On- und ' +
                    'Offline-Welt miteinander verknüpft und ' +
                    'dir ein unvergleichliches Einkaufserlebnis ' +
                    'bietet. Bei uns findest du einen Mix aus ' +
                    'bekannten internationalen Marken und ' +
                    'Produkten von lokalen, regionalen und ' +
                    'nationalen Händlern für jeden Geldbeutel. ' +
                    'Mit aktuell über 1 Million Artikeln und ' +
                    'mehr als 500 Händlern bieten wir dir ' +
                    'ein ständig wachsendes und vielfältiges ' +
                    'Sortiment, das keine Wünsche offen lässt. ' +
                    'Wir sind ein junges, dynamisches Team mit über ' +
                    '180 E-Commerce begeisterten Mitarbeitern ' +
                    '(siroopies genannt). Wir geben jeden Tag ' +
                    'alles, um dir ein Happy Shopping ' +
                    'Erlebnis zu ermöglichen.',
					tag: 'siroop',
				},
				{
					question: 'Was meint ihr mit "Online-Marktplatz"?',
					answer: 'Marktplatz bedeutet, dass bei ' +
                    'uns über 500 Händler angeschlossen sind, ' +
                    'die ihre Produkte anbieten. Für dich hat ' +
                    'ein Marktplatz den Vorteil, dass du ' +
                    'Produkte von verschiedenen Händlern und ' +
                    'Herstellern bei uns vergleichen und direkt ' +
                    'an einem Ort bestellen kannst. Egal ob du ' +
                    'Kleider, Spielzeug oder Technik kaufen ' +
                    'möchtest: du benötigst nur ein Kundenkonto, ' +
                    'erhältst eine Rechnung und hast einen ' +
                    'Ansprechpartner bei Problemen.',
					tag: 'siroop',
				},
				{
					question: 'Wie bestelle ich bei euch?',
					answer: 'Du kannst deine gewünschten ' +
                    'Artikel einfach mittels unserer Suchfunktion ' +
                    'finden, oder du stöberst durch unser Sortiment. ' +
                    'Mit Klick auf "in den Warenkorb" gelangt das ' +
                    'Produkt in deinen Warenkorb. Vom Warenkorb ' +
                    'aus kannst du nun entweder "als Gast" ' +
                    '(ohne Kundenkonto) oder in deinem "Kundenkonto" ' +
                    'angemeldet die Bestellung abschliessen. ' +
                    'Du wählst eine Zahlungsart ' +
					'(siehe <a class="js-t-faq__link" href="#bezahlung-&-gutschriften">Bezahlung ' +
                    '& Gutschriften</a>) und eine Lieferadresse in der ' +
                    'Schweiz aus. Nach erfolgter Bestellung erhältst ' +
                    'du die Bestellbestätigung und deine Rechnung ' +
                    'via E-Mail-Adresse. Sobald dein Produkt ' +
                    'versandbereit ist, bekommst du einen ' +
                    'Tracking-Link, damit du den Lieferstatus ' +
                    'verfolgen kannst. Falls die Produkte deine ' +
                    'Erwartungen nicht erfüllen, hast du die ' +
                    'Möglichkeit diese innerhalb von 14 Tagen ' +
                    'kostenlos zu retournieren. Happy Shopping!',
					tag: 'siroop',
				},
				{
					question: 'Was ist der "Wunschzettel"?',
					answer: 'Mit einem Klick auf das Herzsymbol ' +
                    'kannst du jeden Artikel deinem Wunschzettel ' +
                    'hinzufügen. Dies erleichtert dir die Suche ' +
                    'nach den für dich passenden Produkten; ' +
                    'zudem siehst du die Artikel übersichtlich ' +
                    'aufgelistet. Vom Wunschzettel aus kannst du ' +
                    'die Artikel bequem in den Warenkorb legen.',
					tag: 'siroop',
				},
				{
					question: 'Was kostet die Lieferung?',
					answer: 'Ab einem Bestellwert von ' +
                    'CHF 50.– liefern wir versandkostenfrei. ' +
                    'Bei einem Bestellwert unter CHF 50.– ' +
                    'verrechnen wir dir CHF 7.– Versandkosten.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Wie schnell liefert ihr?',
					answer: 'Wir liefern dir deine ' +
                    'Bestellung an eine gültige Postadresse ' +
                    'in der Schweiz. Bitte beachte, dass ' +
                    'eine Lieferung an die siroop Pick-up ' +
                    'Stationen aktuell nicht mehr möglich ist.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Wohin liefert ihr?',
					answer: 'Ab einem Bestellwert von ' +
                    'CHF 50.– liefern wir versandkostenfrei. ' +
                    'Bei einem Bestellwert unter CHF 50.– ' +
                    'verrechnen wir dir CHF 7.– Versandkosten.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Wo bleibt meine Lieferung?',
					answer: 'Sollte deine Lieferung nach ' +
                    '5 Arbeitstagen noch nicht bei dir ' +
                    'eingetroffen sein, kontaktiere uns ' +
                    'bitte direkt, damit wir den Grund für ' +
                    'den Lieferverzug für dich abklären können.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Ich habe keine Bestätigungs ' +
                    'E-Mail von euch erhalten. Ist meine ' +
                    'Bestellung bei euch eingetroffen?',
					answer: 'Bitte überprüfe deinen ' +
                    'Spam-Ordner. Falls du dennoch nichts ' +
                    'findest, bitten wir dich uns zu ' +
                    'kontaktieren, damit wir den Sachverhalt ' +
                    'für dich abklären können.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Kann ich meine Bestellung ' +
                    '(Produkte, Adresse, Zahlungsart, etc.) ' +
                    'nachträglich noch anpassen?',
					answer: 'Aus technischen Gründen sind ' +
                    'nach erfolgter Bestellung momentan keine ' +
                    'Anpassungen möglich. Bitte kontaktiere ' +
                    'in diesem Fall direkt den Kundenservice.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Kann ich meine Bestellung stornieren?',
					answer: 'Leider ist eine nachträgliche ' +
                    'Stornierung deiner Bestellung nicht ' +
                    'möglich, da sie in der Regel sofort ' +
                    'vom jeweiligen Händler verarbeitet wird. ' +
                    'Du kannst jedoch direkt nach Warenerhalt ' +
                    'eine Retoure anmelden (siehe ' +
                    '<a class="js-t-faq__link" href="#retouren">' +
                    'Retouren</a>) ' +
                    'und die Ware zurücksenden. Bitte beachte, ' +
                    'dass wir anfallende Lieferkosten für ' +
                    'Bestellungen unter CHF 50.- leider nicht ' +
                    'zurückerstatten können.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Ich habe einen falschen ' +
                    'Artikel erhalten, was kann ich tun?',
					answer: 'Bitte sende uns ein Foto ' +
                    'des falsch erhaltenen Artikels an ' +
                    '<a href="mailto:kundenservice@siroop.ch">' +
                    'kundenservice@siroop.ch</a>. Du musst uns ' +
                    'die Ware noch nicht zurückschicken und ' +
                    'auch keine Retoure anmelden. Wir werden ' +
                    'umgehend den Umtausch für dich einleiten.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Ich habe nicht alle Artikel ' +
                    'meiner Bestellung erhalten. Was kann ich tun?',
					answer: 'Falls du Produkte von ' +
                    'mehreren Händlern bestellt hast, ' +
                    'kann es sein, dass diese nicht ' +
                    'gleichzeitig eintreffen. Bitte ' +
                    'überprüfe den Lieferstatus deiner ' +
                    'Bestellung zunächst mit dem Tracking-Link, ' +
                    'den du mit der Versandbestätigung erhalten ' +
                    'hast. Wurde effektiv nicht die vollständige ' +
                    'Stückzahl ausgeliefert, kontaktiere uns ' +
                    'bitte, damit wir die Nachlieferung ' +
                    'umgehend für dich einleiten können.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Mein Artikel ist defekt ' +
                    'angekommen (Fabrikationsfehler). ' +
                    'Was kann ich tun?',
					answer: 'Sämtliche Informationen ' +
                    'dazu findest du unter ' +
                    '<a class="js-t-faq__link" href="#garantie">' +
                    'Garantie</a>.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Mein Paket wurde ' +
                    'von aussen beschädigt. ' +
                    'Was kann ich tun?',
					answer: 'Bitte sende uns Fotos ' +
                    'des beschädigten Artikels sowie ' +
                    'der Aussen- und Innenverpackung ' +
                    'direkt via E-Mail an ' +
                    '<a href="mailto:kundenservice@siroop.ch">' +
                    'kundenservice@siroop.ch</a>. Wir werden ' +
                    'abklären, ob der Schaden bei der Post ' +
                    'oder beim Händler zustande kam. Du ' +
                    'wirst vom Kundenservice über das ' +
                    'weitere Vorgehen informiert.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Ich möchte die Bestellung ' +
                    'nicht behalten. Kann ich sie retournieren?',
					answer: 'Ja, bitte melde innerhalb ' +
                    'von 14 Tagen eine Retoure unter ' +
                    '<a href="https://retoure.siroop.ch/">' +
                    'siroop.ch/retoure</a> an. Weiter ' +
                    'Infos findest du unter ' +
                    '<a class="js-t-faq__link" href="#retouren">' +
                    'Retouren</a>.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Ich habe eine E-Mail ' +
                    'erhalten, dass meine Bestellung  ' +
                    'storniert wurde. Warum?',
					answer: 'In seltenen Fällen kann es ' +
                    'vorkommen, dass ein Verfügbarkeitsproblem ' +
                    'seitens Händler vorliegt. Deine Bestellung ' +
                    'wird dann automatisch storniert. Das tut uns ' +
                    'sehr leid. In diesem Fall wird dir der ' +
                    'Betrag wieder gutgeschrieben. Wir arbeiten ' +
                    'eng mit den Händlern zusammen und setzen ' +
                    'alles daran, diese Fälle möglichst ' +
                    'gering zu halten.',
					tag: 'bestellung-&-lieferung',
				},
				{
					question: 'Was ist eine Retoure?',
					answer: 'Bei uns kannst du deine ' +
                    'Bestellung innerhalb von 14 Tagen ab ' +
                    'Lieferung ohne Angabe von Gründen kostenfrei ' +
                    'zurücksenden, sofern diese keinen Schaden ' +
                    'oder Defekt aufweist. Sobald der Händler ' +
                    'deine Retoure erhalten hat, erhältst du ' +
                    'via Mail eine Bestätigung mit dem ' +
                    'Gutschriftsbetrag. Danach erfolgt die ' +
                    'Gutschrift auf dein Zahlungsmittel resp. ' +
                    'der Artikel wird aus deiner Rechnung ausgebucht.',
					tag: 'retoure',
				},
				{
					question: 'Wie kann ich meine ' +
                    'Bestellung retournieren?',
					answer: 'Melde deine Retoure ganz einfach ' +
                    'innerhalb von 14 Tagen nach Zustellung in ' +
                    'unserem <a href="https://retoure.siroop.ch/">' +
                    'Retourenportal</a> an. Nach erfolgreicher ' +
                    'Anmeldung erhältst du ein vorfrankiertes und ' +
                    'voradressiertes Retourenlabel inklusive einer ' +
                    'kurzen Anleitung. Bitte beachte ' +
                    'unsere Rückgabebedingungen.',
					tag: 'retoure',
				},
				{
					question: 'Wohin muss ich meine Retoure senden?',
					answer: 'Bitte melde deine Retoure immer ' +
                    'in unserem <a href="https://retoure.siroop.ch/">' +
                    'Retourenportal</a> an. Nach ' +
                    'erfolgreicher Anmeldung erhältst du ' +
                    'ein vorfrankiertes und voradressiertes ' +
                    'Retourenlabel inklusive einer kurzen ' +
                    'Anleitung. Wenn du deine Retoure nicht ' +
                    'am Retourenportal anmeldest, können wir ' +
                    'den Gutschriftsbetrag leider nicht ' +
                    'automatisch gutschreiben.',
					tag: 'retoure',
				},
				{
					question: 'Kann ich jeden Artikel innert 14 Tagen retournieren?',
					answer: '<table class="t-faq__table">' +
                    '<tr><td class="t-faq__table-cell t-faq__table-case"><b>Zustand der Ware</b></td>' +
                    '<td class="t-faq__table-cell t-faq__table-case"><b>Erstattungswert</b></td>' +
                    '</tr><tr><td class="t-faq__table-cell t-faq__table-case"><b>UNGEÖFFNET:</b> ' +
                    '<ul><li>Originalverpackte und versiegelte Produkte*</li>' +
                    '<li>Bekleidung und Schuhe: Anprobiert, nicht ' +
                    'getragen und ohne Gebrauchsspuren</li></ul></td>' +
                    '<td class="t-faq__table-cell t-faq__table-solution">Der vollständige ' +
                    'Warenwert wird dir zurückerstattet.</td></tr>' +
                    '<tr><td class="t-faq__table-cell t-faq__table-case"><b>GEÖFFNET:</b> ' +
                    '<ul><li>Produkt ist unbenutzt und OHNE Gebrauchsspuren</li>' +
                    '<li>Originalverpackung geöffnet, aber unbeschädigt vorhanden</li>' +
                    '<li>Mitgeliefertes Zubehör ist vollständig vorhanden</li></ul>' +
                    '<b>Ausgeschlossene Produktgruppen:</b>' +
                    '<ul><li>geöffnete Datenträger und Software</li></ul></td>' +
                    '<td class="t-faq__table-cell t-faq__table-solution">Mindestens 90% ' +
                    'vom Warenwert werden dir zurückerstattet.</td></tr>' +
                    '<tr><td class="t-faq__table-cell t-faq__table-case"><b>GEÖFFNET MIT GEBRAUCHSSPUREN:</b> ' +
                    '<ul><li>Originalverpackung vorhanden</li><li>Produkt weist Gebrauchsspuren auf</li>' +
                    '<li>Mitgeliefertes Zubehör ist vollständig vorhanden</li></ul></td>' +
                    '<td class="t-faq__table-cell t-faq__table-solution">Der Erstattungswert ist abhängig ' +
                    'vom Zustand der Ware und wird vom Händler festgelegt.</td></tr>' +
                    '<tr><td class="t-faq__table-cell t-faq__table-case"><b>BESCHÄDIGTE PRODUKTE:</b> ' +
                    '<ul><li>Produkt ist beschädigt</li>' +
                    '<li>Zubehör ist unvollständig</li><li>Originalverpackung ist nicht vorhanden</li></ul></td>' +
                    '<td class="t-faq__table-cell t-faq__table-negative">Leider können wir ' +
                    'dir in diesem Fall nichts erstatten.</td></tr>' +
                    '<tr><td class="t-faq__table-cell t-faq__table-info" colspan="2">' +
                    '*Generell vom Rückgaberecht ausgeschlossen sind: ' +
                    'Verbrauchsmaterialien, Hygiene- und Kosmetikartikel, ' +
                    'Medikamente, Nahrungsergänzungsmittel, Lebensmittel und Getränke ' +
                    '(Liste nicht abschliessend). Die Versandkosten der Lieferung werden nicht erstattet.</td></tr>' +
                    '<tr><td class="t-faq__table-cell t-faq__table-info" colspan="2">' +
                    '<b>Der Rückversand deiner Retoure ist bei siroop immer kostenfrei.</b></td></tr></table>',
					tag: 'retoure',
				},
				{
					question: 'Kann ich meine Bestellung auch ' +
                    'nach der 14-tägigen Retourenfrist zurücksenden?',
					answer: 'Leider können wir aufgrund unserer ' +
                    'Händlervereinbarungen keine Retouren nach ' +
                    'Ablauf der Retourenfrist akzeptieren.',
					tag: 'retoure',
				},
				{
					question: 'Ich habe eine Retoure angemeldet, ' +
                    'möchte den Artikel nun aber doch behalten. Was kann ich tun?',
					answer: 'Wenn du deine Retoure noch nicht ' +
                    'versendet hast, verfällt die Retourenanmeldung ' +
                    'nach einer gewissen Zeit automatisch. Du musst somit ' +
                    'nichts unternehmen. Solltest du sie bereits versendet ' +
                    'haben, kontaktiere uns bitte.',
					tag: 'retoure',
				},
				{
					question: 'Ich habe einen falschen Artikel ' +
                    'erhalten, muss ich hierfür eine Retoure anmelden?',
					answer: 'Nein, hierbei handelt es sich um einen ' +
                    'Umtausch infolge Falschlieferung. Sämtliche Infos ' +
                    'findest du unter <a class="js-t-faq__link" ' +
                    'href="#bestellung-&-lieferung">' +
                    'Bestellung & Lieferung</a>.',
					tag: 'retoure',
				},
				{
					question: 'Ich habe einen defekten Artikel erhalten, ' +
                    'muss ich hierfür eine Retoure anmelden?',
					answer: 'Nein, hierbei handelt es sich um ' +
                    'einen Garantiefall. Sämtliche Infos findest du unter ' +
                    '<a class="js-t-faq__link" href="#garantie">Garantie</a>.',
					tag: 'retoure',
				},
				{
					question: 'Meine Bestellung ist beschädigt ' +
                    'angekommen, muss ich hierfür eine Retoure anmelden?',
					answer: 'Nein, hierbei handelt es sich um ' +
                    'einen Lager- oder Transportschaden. Sämtliche ' +
                    'Infos findest du unter <a class="js-t-faq__link" ' +
                    'href="#bestellung-&-lieferung">' +
                    'Bestellung & Lieferung</a>.',
					tag: 'retoure',
				},
				{
					question: 'Ich habe eine E-Mail erhalten, dass ' +
                    'meine Retoure angenommen wurde. Jedoch bin ich mit ' +
                    'der Höhe der Gutschrift nicht einverstanden. Was kann ich tun?',
					answer: 'Bitte kontaktiere uns, damit wir eine ' +
                    'Lösung zwischen dir und dem Händler finden können.',
					tag: 'retoure',
				},
				{
					question: 'Ich habe einen Artikel ohne Retourenanmeldung ' +
                    'zurückgesendet. Wann erhalte ich meine Gutschrift?',
					answer: 'Bitte kontaktiere uns, damit wir die ' +
                    'Retoure noch nachträglich für dich anmelden können ' +
                    'und wir dir den Betrag gutschreiben können.',
					tag: 'retoure',
				},
				{
					question: 'Erhalte ich bei einer Retoure auch ' +
                    'die Liefergebühren von CHF 7.- zurückerstattet?',
					answer: 'Da die Anmeldung einer Retoure auf ' +
                    'eigenen Wunsch erfolgt, können wir dir die ' +
                    'Liefergebühren von CHF 7.- bei einem Bestellwert ' +
                    'unter CHF 50.- leider nicht zurückerstatten.',
					tag: 'retoure',
				},
				{
					question: 'Was ist mit Garantie gemeint?',
					answer: 'Sollte dein Artikel defekt ausgeliefert ' +
                    'worden sein (Fabrikationsfehler) oder innerhalb ' +
                    '2 Jahre nach Erhalt ohne Fremdeinwirkung einen ' +
                    'Defekt aufweisen, fällt dieser in die Garantiezeit. ' +
                    'Du kannst uns in diesem Fall einfach kontaktieren, ' +
                    'worauf wir dir eine Postetikette und ein Garantieformular ' +
                    'zum Anmelden des Garantiefalles per E-Mail senden ' +
                    'werden. Der Händler prüft schliesslich, ob der Artikel ' +
                    'einen Anspruch auf Reparatur unter Garantie aufweist. ' +
                    'Nachdem dein Artikel repariert wurde, erfolgt die ' +
                    'Lieferung wieder an dich.',
					tag: 'garantie',
				},
				{
					question: 'Kann ich auch eine Retoure anmelden, ' +
                    'um euch meinen defekten Artikel zurückzuschicken?',
					answer: 'Nein, eine Retoure kann nur bei Produkten ' +
                    'angemeldet werden, die keinen Schaden aufweisen. Bei ' +
                    'defekten Produkten handelt es sich in der Regel um ' +
                    'einen Garantiefall. Bitte kontaktiere uns einfach, ' +
                    'damit wir für dich einen Garantiefall eröffnen können.',
					tag: 'garantie',
				},
				{
					question: 'Ihr habt für mich einen Garantiefall ' +
                    'eröffnet, wie lange dauert die Reparatur noch?',
					answer: 'Die Reparaturzeit unter Garantiezeit ' +
                    'beträgt abhängig vom Händler ca. 2 bis 6 Wochen.',
					tag: 'garantie',
				},
				{
					question: 'Kann ich den Status meines ' +
                    'Garantiefalles selber einsehen?',
					answer: 'Leider ist dies aktuell nicht möglich. ' +
                    'Wenn du deinen Artikel nach 5 Wochen noch nicht ' +
                    'wieder erhalten hast, kannst du dich gerne bei ' +
                    'uns melden. Wir klären den Status dann gerne ' +
                    'für dich ab.',
					tag: 'garantie',
				},
				{
					question: 'Welche Produkte können nicht ' +
                    'unter Garantie repariert werden?',
					answer: 'Ausgeschlossen von der Garantie ' +
                    'sind Gebrauchtwaren, insbesondere Verschleissteile, ' +
                    'Batterien, etc. Die Verkäufer leisten für ihre ' +
                    'Waren und Leistungen eine Garantie von zwei Jahren. ' +
                    'Durch die Garantie wird die gesetzliche ' +
                    'Gewährleistungspflicht nicht eingeschränkt. Der ' +
                    'Entscheid über die Art der Mangelbehebung ' +
                    'liegt beim Verkäufer.',
					tag: 'garantie',
				},
				{
					question: 'Wie kann ich bei euch bezahlen?',
					answer: 'Bei uns kannst du mit <b>Kreditkarte</b>, ' +
                    '<b>PostFinance</b>, <b>Paypal</b> oder auf ' +
                    'Rechnung bezahlen, sofern die Rechnungsoption ' +
                    'für dich zur Verfügung steht.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Weshalb kann ich nicht via Rechnung bezahlen?',
					answer: 'Die Freigabe der Rechnungsoption ' +
                    'erfolgt extern nach automatischer Prüfung der ' +
                    'Eingabedaten durch unseren Finanzdienstleister ' +
                    'Byjuno AG. Deshalb haben wir leider keinen Einfluss ' +
                    'darauf. Weitere Infos findest du unter <a href="https://www.byjuno.ch">www.byjuno.ch</a>.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Ich habe keine Rechnung erhalten. ' +
                    'Könnt ihr sie mir erneut senden?',
					answer: 'Wir versenden die Rechnung immer ' +
                    'per E-Mail. Bitte überprüfe auch deinen ' +
                    'Spam-Ordner. Falls du sie dennoch nicht ' +
                    'findest, bitten wir dich uns zu kontaktieren, ' +
                    'damit wir den Sachverhalt für dich abklären können.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Wer ist Byjuno?',
					answer: 'Byjuno ist unser Partner für ' +
                    'Zahlungen auf Rechnung. Byjuno ist verantwortlich ' +
                    'für die Zahlungsabwicklung, das Forderungsmanagement ' +
                    'sowie das Inkassowesen. Weitere Infos findest du unter ' +
                    '<a href="https://www.byjuno.ch">www.byjuno.ch</a>.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Wie lange habe ich Zeit, um die Rechnung zu bezahlen?',
					answer: 'Du kannst die Rechnung innerhalb ' +
                    'von 20 Tagen ab Erhalt begleichen.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Was passiert, wenn ich die ' +
                    'Rechnung nicht pünktlich bezahle?',
					answer: 'Nach 20 Tagen erhältst du erneut ' +
                    'eine Zahlungserinnerung per E-Mail. Wenn du ' +
                    'die Rechnung auch danach nicht zahlst, erfolgt ' +
                    'die erste Mahnung. Wir behalten uns das Recht vor, ' +
                    'dir auf der Mahnung zusätzliche Mahngebühren in ' +
                    'Rechnung zu stellen.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Kann ich die Rechnung auch in Raten bezahlen?',
					answer: 'Ja, wir bieten dir die Möglichkeit, ' +
                    'deine Rechnung in 3 bequemen Monatsraten zu bezahlen. ' +
                    'Bitte überweise in diesem Fall den gewünschten Teilbetrag, ' +
                    'jedoch mindestens 10% vom Gesamtbetrag. Auf den Restbeträgen ' +
                    'werden Zinskosten und Gebühren gemäss den AGBs unseres ' +
                    'Partners Byjuno AG erhoben. Weitere Infos findest ' +
                    'du unter <a href="https://www.byjuno.ch">www.byjuno.ch</a>.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Ich brauche eine Rechnung in Papierform mit ' +
                    'Einzahlungsschein. Was kann ich tun?',
					answer: 'Auf der Rechnung von Byjuno findest du ' +
                    'einen Link, um einen Einzahlungsschein zu bestellen. ' +
                    'Bitte beachte, dass dir bei dieser Option für jeden ' +
                    'Postversand zu dieser Rechnung von unserem Rechnungspartner ' +
                    'Byjuno AG zusätzlich CHF 3.50 verrechnet werden.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Auf der Rechnung finde ich nur meine ' +
                    'Bestell-Nr., wie sehe ich aber, was ich bestellt habe?',
					answer: 'Mit Klick auf den Link neben der Bestell-Nr. ' +
                    '("Originalrechnung vom Händler") siehst du, was du ' +
                    'bestellt hast. Falls du ein Kundenkonto hast, kannst ' +
                    'du dies auch dort nachschauen.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Meine Bestellung wurde storniert. ' +
                    'Wann erhalte ich die Gutschrift?',
					answer: 'Du erhältst die Gutschrift in der Regel ' +
                    'innert weniger Arbeitstage auf dein Zahlungsmittel ' +
                    'resp. der Artikel wird dann aus deiner Rechnung ausgebucht.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Meine Bestellung wurde storniert, ' +
                    'ich habe aber bereits bezahlt. Wie erhalte ich die Gutschrift?',
					answer: 'Bitte wende dich direkt an Byjuno mit ' +
                    'Angabe deiner Bankdaten. Die E-Mail-Adresse ' +
                    'lautet <a href="mailto:customer-care@byjuno.ch">' +
                    'customer-care@byjuno.ch</a>.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Ich habe einen Artikel retourniert. ' +
                    'Wann erhalte ich die Gutschrift?',
					answer: 'Sobald der Händler deine Retoure ' +
                    'erhalten hat, erhältst du via E-Mail eine Bestätigung ' +
                    'mit dem Gutschriftsbetrag. Danach erfolgt die Gutschrift ' +
                    'auf dein Zahlungsmittel resp. der Artikel wird dir auf ' +
                    'deiner Rechnung gutgeschrieben.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Auf meiner Rechnung fehlt eine Gutschrift ' +
                    'für eine Retoure oder für einen stornierten Artikel. ' +
                    'Was kann ich tun?',
					answer: 'Bitte kontaktiere uns, damit wir den ' +
                    'Sachverhalt für dich abklären können.',
					tag: 'bezahlung-&-gutschriften',
				},
				{
					question: 'Allgemeines',
					answer: 'Du bist ein Teil von siroop! ' +
                    'Fühl dich frei auf unseren Seiten deine Beiträge und Fotos zu posten, ' +
                    'Kommentare zu schreiben und bei unseren Fan-Aktionen mitzumachen.<br><br>' +
                    'Deine Beiträge und Kommentare sind uns wichtig. ' +
                    'Natürlich darfst du auch kritisch deine Meinung äussern. ' +
                    'Dazu findest du unten einige kleine Regeln, die du einhalten solltest.<br><br>' +
                    'Folgendes tolerieren wir nicht:<br>' +
                    '<ul><li>Beleidigende, bedrohende, zu Gewalt auffordernde, diskriminierende, rassistische, ' +
                    'anstössige, obszöne, pornographische, Dritte herabsetzende, ' +
                    'irreführende und gesetzeswidrige Beiträge</li>' +
                    '<li>Inhalte, die gegen Rechte Dritter (inkl. deren Urheber- ' +
                    'und ihrer Persönlichkeitsrechte) verstossen</li>' +
                    '<li>Beiträge zu teilen, welche direkte oder indirekte Werbung für Drittprodukte enthalten</li>' +
                    '<li>Handlungen, welche das einwandfreie Funktionieren unserer ' +
                    'Social Media Kanäle beeinträchtigen sowie Files hochzuladen, ' +
                    'die schädliche Viren oder ähnliches enthalten</li>' +
                    '<li>Korrekte Quellenangaben, die zu Beiträgen gehören, zu verfälschen oder zu löschen</li></ul>' +
                    '<br>siroop behält sich das Recht vor, Inhalte, welche gegen die ' +
                    'oben aufgeführten Punkte verstossen jeder ' +
                    'Zeit zu entfernen<br><br>Bitte habe Verständnis, dass Beiträge von Fans und Besuchern ' +
                    'unserer Social Media Kanäle nicht unbedingt die Meinung Mitarbeitender und ' +
                    'Geschäftspartner von siroop (siroop AG) vertreten.<br><br>' +
                    'Ohne schriftliches Einverständnis von siroop hast du nicht das Recht, Urheberrechte ' +
                    'und Markenzeichen von siroop zu verwenden.<br><br>' +
                    'Als Follower erklärst du dich mit den Rechten und ' +
                    'Pflichten der Sozialen Medien, welche siroop benutzt, ' +
                    'sowie mit den oben genannten Nutzungsbedingungen, ' +
                    'den siroop AGBs und Datenschutzbestimmungen einverstanden.',
					tag: 'social-media-guidelines',
				},
				{
					question: 'Teilnahme an Wettbewerben',
					answer: 'Momentan haben wir keinen aktiven Wettbewerb.',
					tag: 'social-media-guidelines',
				},
				{
					question: 'AGB und Datenschutz',
					answer: 'Unter folgenden Links findest du unsere ' +
                    '<a href="/agb">Allgemeinen Geschäftsbedingungen (AGB)</a> ' +
                    'und <a href="/datenschutz">Datenschutzbestimmungen</a>.',
					tag: 'agb',
				},
				{
					question: 'Kontakt',
					answer: '<ul><li><b>Gratishotline:</b>' +
                    '<a href="tel:0 800 747 667">' +
                    ' 0 800 747 667</a><br>' +
                    'Mo - Fr 8-19 Uhr, Sa: 10-12 Uhr</li>' +
                    '<li><b>E-mail:</b>' +
                    '<a href="mailto:kundenservice@siroop.ch">' +
                    ' kundenservice@siroop.ch</a></li></ul>',
					tag: 'kontakt',
				},
			],
		},
	},
};
