// @siroop/kvasir/source/71-online-banners/prospective/prospective.config.js
const prospective = require('../../00-data/online-banner/prospective');

module.exports = {
	title: 'Prospective Banner',
	name: 'prospective',
	default: '300x250',
	variants: [
		{
			name: '300x250',
			context: {
				modifier: '300x250',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 2,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '300px',
				'min-width': '300px',
			},
		},
		{
			name: '336x280',
			context: {
				modifier: '336x280',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 2,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '336px',
				'min-width': '336px',
			},
		},
		{
			name: '320x160',
			context: {
				modifier: '320x160',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 2,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '320px',
				'min-width': '320px',
			},
		},
		{
			name: '160x600',
			context: {
				modifier: '160x600',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 3,
					axis: 'vertical',
				},
			},
			display: {
				'max-width': '160px',
				'min-width': '160px',
			},
		},
		{
			name: '300x600',
			context: {
				modifier: '300x600',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 3,
					axis: 'vertical',
				},
			},
			display: {
				'max-width': '300px',
				'min-width': '300px',
			},
		},
		{
			name: '728x90',
			context: {
				modifier: '728x90',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 4,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '728px',
				'min-width': '728px',
			},
		},
		{
			name: '800x250',
			context: {
				modifier: '800x250',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 4,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '800px',
				'min-width': '800px',
			},
		},
		{
			name: '970x250',
			context: {
				modifier: '970x250',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 4,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '970px',
				'min-width': '970px',
			},
		},
		{
			name: '994x250',
			context: {
				modifier: '994x250',
				data: {
					products: Array.from({ length: 6 }, () => prospective.product),
					slides: 4,
					axis: 'horizontal',
				},
			},
			display: {
				'max-width': '994px',
				'min-width': '994px',
			},
		},
	],
};
