// @siroop/kvasir/source/60-components/66-organisms/price-slider/price-slider.js
const noUiSlider = require('nouislider');

const selectors = {
	minValueInput: '.js-o-price-slider__input-min-value',
	maxValueInput: '.js-o-price-slider__input-max-value',
	priceSlider: '.js-o-price-slider__script',
	ctaButton: '.js-o-filter-panel__footer-button',
};

class PriceSlider {
	constructor($el) {
		this.minValueInput = $el.find(selectors.minValueInput).get(0);
		this.maxValueInput = $el.find(selectors.maxValueInput).get(0);
		this.priceSlider = $el.find(selectors.priceSlider).get(0);
		this.ctaButton = $el.find(selectors.ctaButton).get(0);

		this.startMinValue = parseInt(this.minValueInput.value, 10);
		this.startMaxValue = parseInt(this.maxValueInput.value, 10);

		this.rangeMin = parseInt(this.minValueInput.getAttribute('data-range-min'), 10);
		this.rangeMax = parseInt(this.maxValueInput.getAttribute('data-range-max'), 10);
		this.sameRange = false;

		// Fix because noUISlider does not support two equal values for the range property
		if (this.rangeMin === this.rangeMax) {
			this.sameRange = true;
			this.rangeMin = this.rangeMin - 1;

			if (this.sameRange) {
				this.priceSlider.setAttribute('disabled', true);
			}
		}

		this.initialConfiguration = {
			start: [this.startMinValue, this.startMaxValue],
			connect: true,
			range: {
				min: this.rangeMin,
				max: this.rangeMax,
			},
			cssPrefix: 'o-price-slider__script-',
		};

		this.noUISlider = noUiSlider.create(this.priceSlider, this.initialConfiguration);

		this.priceSlider.noUiSlider.on('update', this.onSliderUpdate.bind(this));

		this.minValueInput.addEventListener('change', (ev) => {
			this.priceSlider.noUiSlider.set([ev.target.value, null]);
		});

		this.maxValueInput.addEventListener('change', (ev) => {
			this.priceSlider.noUiSlider.set([null, ev.target.value]);
		});
	}

	onSliderUpdate(values, handle) {
		const value = values[handle];
		if (handle) {
			this.maxValueInput.value = Math.round(value);
		} else {
			this.minValueInput.value = Math.round(value);
		}
	}

	onSliderButtonClick() {
		this.minValueInput.setAttribute('value', this.minValueInput.value);
		this.maxValueInput.setAttribute('value', this.maxValueInput.value);

		this.startMinValue = parseInt(this.minValueInput.value, 10);
		this.startMaxValue = parseInt(this.maxValueInput.value, 10);
	}
}

export default PriceSlider;
