// @siroop/kvasir/source/00-data/product/index.js
const faker = require('faker');

let url = `/components/preview/t-pdp?name=${faker.commerce.productName()}`;
url = url.replace(/\s+/g, '');

const singleProduct = () => ({
	title: faker.commerce.productName(),
	manufacturer: faker.company.companyName(),
	merchant: {
		name: faker.company.companyName(),
		logo: '',
	},
	location: '',
	url,
	id: faker.random.number(),
	category: faker.random.arrayElement(['Computer & Elektronik', 'Lebensmittel & Getränke', 'Mode & Accessoires', 'Baby & Spielzeug']),
	logo: '',
	modifier: '',
	carouselTitle: faker.lorem.words(),
	descriptionShort: faker.lorem.sentence(),
	description: faker.lorem.paragraphs(),
	tealium: {
		'data-utag-impression': '',
		'data-utag': '{"sku": "123-1"}',
	},
	images: [
		{
			srcset: [
				{
					src: '/img/content/product/iphone/iphone-200.jpg',
					width: '1x',
				},
				{
					src: '/img/content/product/iphone/iphone-400.jpg',
					width: '2x',
				},
			],
			src: '/img/content/product/iphone/iphone-200.jpg',
			srcsetString: '/img/content/product/iphone/iphone-200.jpg 1x, /img/content/product/iphone/iphone-400.jpg 2x',
			alt: faker.commerce.productName(),
			sizes: '',
		},
		{
			srcset: [
				{
					src: '/img/content/product/iphone/iphone-200.jpg',
					width: '1x',
				},
				{
					src: '/img/content/product/iphone/iphone-400.jpg',
					width: '2x',
				},
			],
			src: '/img/content/product/iphone/iphone-200.jpg',
			srcsetString: '/img/content/product/iphone/iphone-200.jpg 1x, /img/content/product/iphone/iphone-400.jpg 2x',
			alt: faker.commerce.productName(),
			sizes: '',
		},
		{
			srcset: [
				{
					src: '/img/content/product/iphone/iphone-200.jpg',
					width: '1x',
				},
				{
					src: '/img/content/product/iphone/iphone-400.jpg',
					width: '2x',
				},
			],
			src: '/img/content/product/iphone/iphone-200.jpg',
			srcsetString: '/img/content/product/iphone/iphone-200.jpg 1x, /img/content/product/iphone/iphone-400.jpg 2x',
			alt: faker.commerce.productName(),
			sizes: '',
		},
		{
			srcset: [
				{
					src: '/img/content/product/iphone/iphone-200.jpg',
					width: '1x',
				},
				{
					src: '/img/content/product/iphone/iphone-400.jpg',
					width: '2x',
				},
			],
			src: '/img/content/product/iphone/iphone-200.jpg',
			srcsetString: '/img/content/product/iphone/iphone-200.jpg 1x, /img/content/product/iphone/iphone-400.jpg 2x',
			alt: faker.commerce.productName(),
			sizes: '',
		},
	],
	groups: {
		name: 'Farbe',
		items: [
			{
				name: 'Black',
				url: '/components/preview/t-pdp?type=black',
				disabled: false,
				image: {
					srcset: [
						{
							src: '/img/content/product/iphone/iphone7-black-50x50.jpg',
							width: '50w',
						},
						{
							src: '/img/content/product/iphone/iphone7-black-50x50.jpg',
							width: '100w',
						},
						{
							src: '/img/content/product/iphone/iphone7-black-50x50.jpg',
							width: '200w',
						},
					],
					src: '/img/content/product/iphone/iphone7-black-50x50.jpg',
					alt: 'Black',
					srcsetString: '/img/content/product/iphone/iphone7-black-50x50.jpg 50w, /img/content/product/iphone/iphone7-black-50x50.jpg 100w, /img/content/product/iphone/iphone7-black-50x50.jpg 200w',
					sizes: '',
				},
			},
			{
				name: 'Diamond Black',
				url: '/components/preview/t-pdp?type=diamondblack',
				disabled: false,
				image: {
					srcset: [
						{
							src: '/img/content/product/iphone/iphone7-diamondblack-50x50.jpg',
							width: '50w',
						},
						{
							src: '/img/content/product/iphone/iphone7-diamondblack-50x50.jpg',
							width: '100w',
						},
						{
							src: '/img/content/product/iphone/iphone7-diamondblack-50x50.jpg',
							width: '200w',
						},
					],
					src: '/img/content/product/iphone/iphone7-black-50x50.jpg',
					alt: 'Diamond Black',
					srcsetString: '/img/content/product/iphone/iphone7-diamondblack-50x50.jpg 50w, /img/content/product/iphone/iphone7-diamondblack-50x50.jpg 100w, /img/content/product/iphone/iphone7-diamondblack-50x50.jpg 200w',
					sizes: '',
				},
			},
			{
				name: 'Gold',
				url: '/components/preview/t-pdp?type=gold',
				disabled: true,
				image: {
					srcset: [
						{
							src: '/img/content/product/iphone/iphone7-gold-50x50.jpg',
							width: '50w',
						},
						{
							src: '/img/content/product/iphone/iphone7-gold-50x50.jpg',
							width: '100w',
						},
						{
							src: '/img/content/product/iphone/iphone7-gold-50x50.jpg',
							width: '200w',
						},
					],
					src: '/img/content/product/iphone/iphone7-black-50x50.jpg',
					alt: 'Gold',
					srcsetString: '/img/content/product/iphone/iphone7-gold-50x50.jpg 50w, /img/content/product/iphone/iphone7-gold-50x50.jpg 100w, /img/content/product/iphone/iphone7-gold-50x50.jpg 200w',
					sizes: '',
				},
			},
		],
	},
	variants: {
		name: 'Produkt Material',
		items: [
			{
				is_label: true,
				name: 'Bitte Produktmaterial wählen',
				url: '#',
				selected: 'true',
				disabled: false,
			},
			{
				name: faker.commerce.productMaterial(),
				url: '#',
				selected: false,
				disabled: true,
			},
			{
				name: faker.commerce.productMaterial(),
				url: '#',
				selected: false,
				disabled: false,
			},
			{
				name: faker.commerce.productMaterial(),
				url: '#',
				selected: false,
				disabled: false,
			},
			{
				name: faker.commerce.productMaterial(),
				url: '#',
				selected: false,
				disabled: false,
			},
		],
	},
	mainAttributes: [
		{
			key: 'Hersteller',
			value: faker.company.companyName(),
		},
		{
			key: 'Product Adjective',
			value: faker.commerce.productAdjective(),
		},
		{
			key: 'Product Adjective',
			value: faker.commerce.productAdjective(),
		},
	],
	specifications: [
		{
			key: 'Artikelnummer',
			value: 'A-A339d#33333',
		},
		{
			key: 'Auflösung Display',
			value: '2048px x 1024px',
		},
		{
			key: 'Display Diagonale',
			value: '4.7 Zoll',
		},
		{
			key: 'Farbe',
			value: 'Silber',
		},
		{
			key: 'Hersteller',
			value: 'Apple',
		},
		{
			key: 'Auflösung Kamera',
			value: '5MP',
		},
		{
			key: 'Barometer',
			value: 'Ja',
		},
		{
			key: 'Bildschirmgrösse',
			value: '11.7cm',
		},
		{
			key: 'Breite',
			value: '67mm',
		},
		{
			key: 'Digitaler Zoom',
			value: '3x',
		},
		{
			key: 'Farbe',
			value: 'Gold',
		},
		{
			key: 'Bluetooth',
			value: 'Ja',
		},
	],
	sizes: [
		{
			name: '',
			value: '',
			selected: true,
		},
		{
			name: '',
			value: '',
		},
		{
			name: '',
			value: '',
		},
	],
	colors: [
		{
			name: faker.commerce.color(),
			value: faker.commerce.color(),
			selected: true,
		},
		{
			name: faker.commerce.color(),
			value: faker.commerce.color(),
		},
		{
			name: faker.commerce.color(),
			value: faker.commerce.color(),
		},
	],
	stockStatus: 'zur Zeit nicht verfügbar',
	delivery: {
		spedition: 'Spedition ab Montag, den 27.07.2016',
		pickup: 'Abholung in einer <a href="#">Pick-up Station</a> möglich.',
	},
	shipping: {
		method: '',
		icon: '',
		priceValue: faker.random.number({ min: 5, max: 50 }),
		priceCurrency: 'CHF',
		free: 'gratis',
	},
	availability: {
		minValue: '1',
		maxValue: faker.random.number({ min: 1 }),
		initialValue: '1',
	},
	price: {
		priceUnit: '',
		pricePrefix: 'ab',
		priceValue: faker.commerce.price(),
		priceDiscount: faker.commerce.price(),
		priceCurrency: 'CHF',
		priceSuffix: '',
		unitPrice: 'Stückpreis',
		priceDrop: faker.random.boolean(),
	},
	cheapestPrice: {
		value: faker.commerce.price(),
		currency: 'CHF',
		label: 'ab',
	},
});

const producstList = numProducts => Array.from({ length: numProducts }, singleProduct);

const products = producstList(36);
products[1].availability.maxValue = 0;

module.exports = products;
