// @siroop/kvasir/source/00-data/product/products-cart.js
const productsCart = [
	{
		image: [
			{
				srcset: [
					{
						src: 'https://cdn.siroop.ch/media/images/sized/694a305c78c0a5a9611684b8318dc419.200x200.jpg',
					},
					{
						src: 'https://cdn.siroop.ch/media/images/sized/694a305c78c0a5a9611684b8318dc419.400x400.jpg',
						width: '2x',
					},
				],
				src: 'http://placehold.it/200x200',
			},
		],
		name: 'The North Face Shinpuru Jacket',
		location: '',
		price: {
			pricePrefix: '',
			priceValue: '640.00',
			priceDiscount: '560.00',
			priceCurrency: 'CHF',
		},
		priceCurrency: 'CHF',
		unitPrice: 'Stückpreis',
		url: '#1',
		modifier: 'style-1',
		category: 'Handpicked',
		attributes: [
			{
				key: 'Hersteller',
				value: 'Sony',
			},
			{
				key: 'Formfaktor',
				value: 'In-Ear',
			},
			{
				key: 'Anschlusstechnik',
				value: 'verkabelt',
			},
		],
		availability: {
			minValue: 1,
			maxValue: 20,
			initialValue: 1,
		},
		merchant: {
			name: 'Fust',
			logo: 'http://placehold.it/125x125',
		},
		delivery: {
			spedition: 'Spedition ab Montag, den 27.07.2016',
			pickup: 'Abholung in einer <a href="#">Pick-up Station</a> möglich.',
		},
		shipping: {
			method: 'Versandkosten',
			icon: 'icon-parcel',
			priceValue: '5.00',
			priceCurrency: 'CHF',
		},
	},
	{
		image: [
			{
				srcset: [
					{
						src: 'https://cdn.siroop.ch/media/images/sized/694a305c78c0a5a9611684b8318dc419.200x200.jpg',
					},
					{
						src: 'https://cdn.siroop.ch/media/images/sized/694a305c78c0a5a9611684b8318dc419.400x400.jpg',
						width: '2x',
					},
				],
				src: 'http://placehold.it/200x200',
			},
		],
		name: 'The North Face Shinpuru Jacket',
		location: '',
		price: {
			pricePrefix: '',
			priceValue: '640.00',
			priceDiscount: '',
			priceCurrency: 'CHF',
		},
		priceCurrency: 'CHF',
		unitPrice: 'Stückpreis',
		url: '#2',
		modifier: 'style-1',
		category: 'Handpicked',
		attributes: [
			{
				key: 'Hersteller',
				value: 'Sony',
			},
			{
				key: 'Formfaktor',
				value: 'In-Ear',
			},
			{
				key: 'Anschlusstechnik',
				value: 'verkabelt',
			},
		],
		availability: {
			minValue: 1,
			maxValue: 20,
			initialValue: 1,
		},
		merchant: {
			name: 'Fust',
			logo: 'http://placehold.it/125x125',
		},
		delivery: {
			spedition: 'Spedition ab Montag, den 27.07.2016',
			pickup: 'Abholung in einer <a href="#">Pick-up Station</a> möglich.',
		},
		shipping: {
			method: 'Versandkosten',
			icon: 'icon-parcel',
			priceValue: '5.00',
			priceCurrency: 'CHF',
			free: 'gratis',
		},
	},
	{
		image: [
			{
				srcset: [
					{
						src: 'https://cdn.siroop.ch/media/images/sized/694a305c78c0a5a9611684b8318dc419.200x200.jpg',
					},
					{
						src: 'https://cdn.siroop.ch/media/images/sized/694a305c78c0a5a9611684b8318dc419.400x400.jpg',
						width: '2x',
					},
				],
				src: 'http://placehold.it/200x200',
			},
		],
		name: 'The North Face Shinpuru Jacket',
		location: '',
		price: {
			pricePrefix: '',
			priceValue: '640.00',
			priceDiscount: '',
			priceCurrency: 'CHF',
		},
		priceCurrency: 'CHF',
		unitPrice: 'Stückpreis',
		url: '#3',
		modifier: 'style-1',
		category: 'Handpicked',
		attributes: [
			{
				key: 'Hersteller',
				value: 'Sony',
			},
			{
				key: 'Formfaktor',
				value: 'In-Ear',
			},
			{
				key: 'Anschlusstechnik',
				value: 'verkabelt',
			},
		],
		availability: {
			minValue: 1,
			maxValue: 20,
			initialValue: 1,
		},
		merchant: {
			name: 'Fust',
			logo: 'http://placehold.it/125x125',
		},
		delivery: {
			spedition: 'Spedition ab Montag, den 27.07.2016',
			pickup: 'Abholung in einer <a href="#">Pick-up Station</a> möglich.',
		},
		shipping: {
			method: 'Versandkosten',
			icon: 'icon-parcel',
			priceValue: '5.00',
			priceCurrency: 'CHF',
			free: 'gratis',
		},
	},
];

module.exports = productsCart;
