// @siroop/kvasir/source/00-data/product/product-family.js
const productFamily = [
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-1-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-1-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-1-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-2-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-2-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-2-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmodemodemode',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-3-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-3-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-3-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'M',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-4-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-4-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-4-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: '',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-5-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-5-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-5-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-1-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-1-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-1-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '4511000000 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-2-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-2-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-2-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-3-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-3-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-3-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-4-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-4-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-4-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-5-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-5-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-5-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
			],
			color: '#778288',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
		textColor: '#000',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-1-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-1-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-1-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/a6895cec3686af2f4959d5c8cd707e43.400x400.jpg',
				},
			],
			color: '#1be4bb',
		},
		title: 'Frauenmode Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
		textColor: '#fff',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-2-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-2-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-2-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/870cb9d89d5b4f8a2f97a456e8e32fe4.400x400.jpg',
				},
			],
			color: '#ffb800',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
		textColor: '#000',
	},
	{
		images: {
			background: [
				{
					url: '/img/content/product-families/background-image-3-320x240.jpg',
				},
				{
					url: '/img/content/product-families/background-image-3-767x575.jpg',
				},
				{
					url: '/img/content/product-families/background-image-3-1024x768.jpg',
				},
			],
			featured: [
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
				{
					url: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				},
			],
			color: '#00c5ff',
		},
		title: 'Frauenmode',
		subtitle: '45 Produkte',
		url: '#',
		textColor: '#fff',
	},
];

module.exports = productFamily;
