// @siroop/kvasir/source/60-components/66-organisms/form-checkout/form-checkout.config.js
const formBillingAddress = require('../../../00-data/form/billing-address');
const formShippingAddress = require('../../../00-data/form/shipping-address');
const formPersonalData = require('../../../00-data/form/personal-data');
const formContactInfo = require('../../../00-data/form/contact-info');
const formCreateAccount = require('../../../00-data/form/create-account');

module.exports = {
	title: 'Form Checkout',
	name: 'form-checkout',
	display: {
		'max-width': '600px',
	},
	context: {
		data: {
			labelRequiredFields: '* Pflichtfelder',
			formBillingAddress,
			formCheckShipping: {
				id: 'form-checkout-is-shipping',
				name: 'form-checkout-is-shipping',
				label: 'Abweichende Lieferadresse',
			},
			formShippingAddress,
			formPersonalData,
			formContactInfo,
			formCheckAccount: {
				id: 'form-checkout-is-new-customer',
				name: 'form-checkout-is-new-customer',
				label: 'Ich möchte ein Kundenkonto erstellen',
				benefits: [
					'alle deine bestellungen im Ueberblick',
					'Rechnung downloaden',
				],
			},
			formCreateAccount,
		},
	},
};
