// @siroop/kvasir/source/70-templates/series-page/series-page.config.js
const brandData = require('../../60-components/66-organisms/page-heading/page-heading.config');
const productListData = require('../../60-components/66-organisms/list/list.config');
const productCategory = require('../../00-data/category/');
const videoData = require('../../60-components/64-molecules/video-player/video-player.config');
const gallery = require('../../00-data/gallery/');

const brandDataClone = Object.assign({}, brandData.context.data);
brandDataClone.slogan = '';

module.exports = {
	title: 'Series page',
	context: {
		data: {
			video: videoData.context.data,
			gallery: {
				items: gallery,
				labelNext: 'Next',
				labelPrev: 'Prev',
			},
			breadcrumb: {
				items: [
					{
						url: '#',
						name: 'Startseite',
					},
					{
						url: '#',
						name: 'Marken',
					},
					{
						url: '#',
						name: 'iPhone',
					},
				],
			},
			brand: brandDataClone,
			productList: {
				showMoreItems: 'js-t-series-page-products-show-more',
				items: [
					productListData.context.data.items[0],
					productListData.context.data.items[1],
					productListData.context.data.items[2],
					productListData.context.data.items[3],
					productListData.context.data.items[0],
					productListData.context.data.items[1],
					productListData.context.data.items[2],
					productListData.context.data.items[3],
				],
				moreUrlLabel: '',
				moreUrl: '',
				modifierListItem: 'vertical',
			},
			showMoreButton: {
				initialItems: 6,
				showMoreItems: 'js-t-series-page-products-show-more',
				expandLabel: 'Show more',
				collapseLabel: 'Show less',
			},
			teasers: {
				title: 'Weitere Produktfamilien von Apple',
				rows: [
					{
						modifier: 'style-2',
						items: productCategory.slice(2, 5),
					},
				],
			},
			body: `
				<h1>Content Template Subline</h1>
				<h2>Content Template Subline</h2>
				<h3>Content Template Subline</h3>
				<div class="image-left">
					<div class='image'>
						<picture>
							<img src="/img/content/teaser/bg-teaser-home-large-x1.jpg" alt="">
						</picture>
					</div>
					<div class="text">
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores at aut corporis
						cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
					</div>
				</div>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<ul>
				<li>Lorem Ipsum Doler sit amet.</li>
				<li>Consectetur adipisicing elit</li>
				<li><a href="#alink">Amet architecto asperiores</a></li>
				<li>Perspiciatis saepe sequi temporibus vel</li>
				</ul>
				<div class="image-right">
					<div class='image'>
						<picture>
							<img src="/img/content/teaser/bg-teaser-home-large-x1.jpg" alt="">
						</picture>
					</div>
					<div class="text">
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores at aut corporis
						cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
					</div>
				</div>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<p><img src="/img/content/teaser/bg-teaser-home-large-x1.jpg" alt=""></p>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<ol>
				<li>Lorem Ipsum Doler sit amet.</li>
				<li>Consectetur adipisicing elit</li>
				<li><a href="#alink">Amet architecto asperiores</a></li>
				<li>Perspiciatis saepe sequi temporibus vel</li>
				</ol>
			`,
		},
	},
};
