// @siroop/kvasir/source/80-layouts/default/default.config.js
const uspBanner = require('../../60-components/64-molecules/usp-banner/usp-banner.config');
const header = require('../../60-components/66-organisms/header/header.config');
const footer = require('../../60-components/66-organisms/footer/footer.config');
const footerMeta = require('../../60-components/66-organisms/footer-meta/footer-meta.config');
const bannerMeta = require('../../60-components/64-molecules/banner-meta/banner-meta.config');

module.exports = {
	context: {
		pageTitle: 'Siroop.ch',
		data: {
			showNotification: false,
			uspBannerFooter: uspBanner.variants[1].context.data,
			header: header.context.data,
			footer: footer.context.data,
			footerMeta: footerMeta.context.data,
			bannerMeta: bannerMeta.context.data,
			notification: {
				text: 'Das Produkt wurde erfolgreich zum Warenkorb hinzugefügt.',
				icon: 'checkmark',
				action: {
					text: 'Zum Warenkorb',
					url: '/components/preview/t-cart',
				},
				modifier: 'with-button o-notification--bottom-right',
			},
		},
	},
};
