// @siroop/kvasir/source/60-components/62-atoms/pill/pill.config.js
module.exports = {
	title: 'Pill',
	name: 'pill',
	default: 'default',
	context: {
		data: {
			modifier: 'default a-pill--medium',
			text: 'Pill',
		},
	},
	variants: [
		{
			name: 'toggle',
			context: {
				data: {
					modifier: 'toggle a-pill--medium',
					text: 'Toggle',
				},
			},
		},
		{
			name: 'active',
			context: {
				data: {
					modifier: 'active a-pill--medium',
					text: 'Active',
				},
			},
		},
		{
			name: 'inactive',
			context: {
				data: {
					modifier: 'inactive a-pill--medium',
					text: 'Inactive',
				},
			},
		},
		{
			name: 'small',
			context: {
				data: {
					modifier: 'small',
					text: 'Small',
				},
			},
		},
		{
			name: 'medium',
			context: {
				data: {
					modifier: 'medium',
					text: 'Medium',
				},
			},
		},
		{
			name: 'large',
			context: {
				data: {
					modifier: 'large',
					text: 'Large',
				},
			},
		},
		{
			name: 'round',
			context: {
				data: {
					modifier: 'round',
					text: 'A',
				},
			},
		},
	],
};
