// @siroop/kvasir/source/60-components/64-molecules/form-field-radio/form-field-radio.config.js
module.exports = {
	title: 'Form Field Radio',
	name: 'form-field-radio',
	context: {
		modifier: '',
		data: {
			title: 'Label',
			items: [
				{
					id: 'radio-01',
					name: 'gender',
					value: 'herr',
					label: 'Herr',
					options: {
						type: 'radio',
					},
				},
				{
					id: 'radio-02',
					name: 'gender',
					value: 'frau',
					label: 'Frau',
					options: {
						type: 'radio',
						checked: 'true',
					},
				},
				{
					id: 'radio-03',
					name: 'gender',
					value: 'herr',
					label: 'Herr',
					options: {
						type: 'radio',
						disabled: 'true',
					},
				},
			],
		},
	},
	variants: [
		{
			name: 'row',
			context: {
				modifier: 'row',
			},
		},
		{
			name: 'boxed',
			context: {
				modifier: 'boxed',
				data: {
					items: [
						{
							id: 'radio-01',
							name: 'payment-method',
							value: 'card',
							label: 'Deine gespeicherte Zahlungsarten',
							svgs: [
								'visa-colored',
							],
							text: '**** **** **** 2345<br /> gültig bis: 05/18',
							options: {
								type: 'radio',
								delete: 'Entfernen',
							},
						},
						{
							id: 'radio-02',
							name: 'payment-method',
							value: 'card',
							label: 'Kartenzahlung',
							desc: 'Akzeptierte Zahlungsarten:',
							svgs: [
								'postfinance-colored',
								'visa-colored',
								'mastercard-colored',
								'amex-colored',
							],
							options: {
								type: 'radio',
							},
						},
						{
							id: 'radio-03',
							name: 'payment-method',
							value: 'bill',
							label: 'Rechnung',
							desc: 'Du erhältst von uns per E-Mail eine Rechnung und hast ab Zustellung 30 Tage Zeit, sie zu begleichen.',
							svgs: [
								'bulletin-colored',
							],
							options: {
								type: 'radio',
								disabled: 'false',
							},
						},
						{
							id: 'radio-04',
							name: 'payment-method',
							value: 'other',
							label: 'Other',
							desc: 'Some other payment method',
							options: {
								type: 'radio',
								checked: true,
							},
						},
						{
							id: 'radio-05',
							name: 'payment-method',
							value: 'otherone',
							label: 'Other one',
							options: {
								type: 'radio',
								checked: true,
							},
						},
					],
				},
			},
		},
	],
};
