// @siroop/kvasir/source/60-components/64-molecules/price/price.config.js
module.exports = {
	title: 'Price',
	name: 'price',
	default: 'default',
	variants: [
		{
			name: 'default',
			context: {
				modifier: 'default',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'300.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'abstract',
			context: {
				modifier: 'abstract',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'small',
			context: {
				modifier: 'small',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'large',
			context: {
				modifier: 'large',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'x-large',
			context: {
				modifier: 'x-large',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'xx-large',
			context: {
				modifier: 'xx-large',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'larger',
			context: {
				modifier: 'larger',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'reduced',
			context: {
				modifier: 'reduced',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'free',
			context: {
				modifier: 'free',
				data: {
					priceValue: '0.00',
					priceCurrency: 'CHF',
				},
			},
		},
		{
			name: 'price-drop',
			context: {
				modifier: 'price-drop m-price--x-large',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
					priceDrop: true,
				},
			},
		},
		{
			name: 'striked',
			context: {
				modifier: 'striked',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
		{
			name: 'pdp-pricedrop-example',
			context: {
				modifier: 'larger m-price--reduced m-price--icon',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
					icon: 'pricedrop',
					iconModifier: 'large',
				},
			},
		},
		{
			name: 'responsive',
			context: {
				modifier: 'responsive',
				data: {
					pricePrefix: 'ab',
					priceValue: '10\'100.00',
					priceCurrency: 'CHF',
					priceSuffix: '',
				},
			},
		},
	],
};
