// @siroop/kvasir/source/70-templates/search-results/search-results.config.js
const productsData = require('../../00-data/product/');
const listFooter = require('../../60-components/66-organisms/list-footer/list-footer.config');
const listHeader = require('../../60-components/66-organisms/list-header/list-header.config');
const linkList = require('../../60-components/66-organisms/link-list/link-list.config');
const categoryData = require('../../60-components/66-organisms/category-list/category-list.config');

module.exports = {
	title: 'Search results',
	name: 'search-results',
	context: {
		modifier: '',
		modifierListItem: 'horizontal',
		modifierPrice: 'responsive',
		filtersLabel: 'Filter',
		categoryList: categoryData.context.data,
		panelCategories: {
			name: 'Kategorien',
			back: 'Kategorien',
			buttons: [
				{
					name: 'Zurück zu den Suchergebnissen',
					modifiers: ['borders'],
					classes: ['js-o-filter-panel__back'],
				},
			],
			triggerButton: {
				class: 't-search-results__category-button',
			},
			url: '/components/preview/t-category-l3',
			activeCategories: {
				name: 'Kategorien',
				items: linkList.context.data.items,
			},
		},
		panelFilters: {
			name: 'Filter',
			back: 'Ergebnisse für Fahrrad',
			buttons: [
				{
					name: 'Abbrechen',
					modifiers: ['borders', 'small'],
					classes: ['js-o-filter-panel__back'],
				},
				{
					name: '156 Ergebnisse anzeigen',
					modifiers: ['filled'],
				},
			],
			triggerButton: {
				class: 't-search-results__filter-button',
			},
			activeFilters: {
				resetFilter: 'Alle Filter zurücksetzen',
				items: [
					{
						text: 'Marke: <b>Blackberry</b>',
					},
					{
						text: 'Farbe: <b>Blau</b>',
					},
					{
						text: 'Farbe: <b>Blau</b>',
					},
					{
						text: 'Farbe: <b>Blau</b>',
					},
				],
			},
			items: [
				{
					name: 'Preis',
					value: 'checked',
					back: 'Filter',
					buttons: [
						{
							name: 'Anwenden',
							modifiers: ['filled'],
						},
					],
					active: true,
					content: {
						name: 'Preis',
						component: {
							name: 'o-price-slider',
							data: {
								id: '1',
								currency: 'CHF',
								button: {
									text: 'Anwenden',
								},
								start: {
									min: 5,
									max: 880,
								},
								range: {
									min: 5,
									max: 880,
								},
								label: {
									from: 'Von',
									to: 'Bis',
								},
							},
						},
					},
				},
				{
					name: 'Marke',
					value: 'checked',
					buttons: [
						{
							name: 'Anwenden',
							modifiers: ['filled'],
						},
					],
					active: true,
					content: {
						name: 'Marke',
						component: {
							name: 'o-checkbox-list',
							data: {
								items: [
									{
										id: 'checkbox-01',
										name: 'checkbxox',
										label: 'Panasonic',
									},
									{
										id: 'checkbox-02',
										name: 'checkbox',
										label: 'Lego',
									},
									{
										id: 'checkbox-03',
										name: 'checkbox',
										label: 'Braun',
									},
									{
										id: 'checkbox-04',
										name: 'checkbox',
										label: 'Samsung',
									},
									{
										id: 'checkbox-05',
										name: 'checkbox',
										label: 'Bosch',
									},
									{
										id: 'checkbox-06',
										name: 'checkbox',
										label: 'Hugo Boss',
									},
									{
										id: 'checkbox-07',
										name: 'checkbox',
										label: 'Apple',
									},
									{
										id: 'checkbox-08',
										name: 'checkbox',
										label: 'LG',
									},
									{
										id: 'checkbox-09',
										name: 'checkbxox',
										label: 'Acer',
									},
									{
										id: 'checkbox-10',
										name: 'checkbox',
										label: 'Asus',
									},
									{
										id: 'checkbox-11',
										name: 'checkbox',
										label: 'Puma',
									},
									{
										id: 'checkbox-12',
										name: 'checkbox',
										label: 'Silvio Tossi',
									},
									{
										id: 'checkbox-13',
										name: 'checkbox',
										label: 'Yamaha',
									},
									{
										id: 'checkbox-14',
										name: 'checkbox',
										label: 'Boss',
									},
									{
										id: 'checkbox-15',
										name: 'checkbox',
										label: 'Chanel',
									},
									{
										id: 'checkbox-16',
										name: 'checkbox',
										label: 'Dior',
									},
								],
								button: 'Anwenden',
							},
						},
					},
				},
			],
		},
		search: {
			term: 'Fahrrad',
			termLabel: 'Ergebnisse für',
			message: 'Leider gab es keine passende Treffer.',
			messages: [
				'Kontrolliere die Schreibweise oder',
				'Benutze die jeweiligen Kategorien',
			],
			error: {
				message: 'Keine Ergebnisse für',
				showAllProducts: 'Alle Produkte anzeigen',
				back: 'zurück',
			},
			results: {
				listHeader: listHeader.context.data,
				listFooter: listFooter.context.data,
				items: productsData,
				modifierListItem: 'horizontal',
			},
		},
		categories: {
			title: 'Unser Sortiment',
			items: [
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
				{
					name: 'Baby & Spielzeug',
					badge: '56798',
					url: '#',
				},
			],
		},
	},
};
