// @siroop/kvasir/source/70-templates/teaser-list-usage/teaser-list-usage.config.js
const productCategory = require('../../00-data/category/');

module.exports = {
	title: 'Teaser List Usage',
	name: 'teaser-list-usage',
	status: 'prototype',
	context: {
		data: {
			rows: [
				{
					modifier: 'style-1',
					items: productCategory.slice(0, 2),
				},
			],
			expandButtonLabel: 'Alle Produktfamilien anzeigen',
		},
	},
	variants: [
		{
			name: 'with-expand-button',
			context: {
				data: {
					rows: [
						{
							modifier: 'style-1',
							items: productCategory.slice(0, 2),
						},
						{
							expand: true,
							expandLabel: 'Alle Produktfamilien anzeigen',
							modifier: 'style-2',
							items: productCategory.slice(0, 3),
						},
					],
				},
			},
		},
		{
			name: 'two-items',
			context: {
				data: {
					rows: [
						{
							modifier: 'style-1',
							items: productCategory.slice(0, 2),
						},
					],
				},
			},
		},
		{
			name: 'three-items',
			context: {
				data: {
					rows: [
						{
							modifier: 'style-2',
							items: productCategory.slice(0, 3),
						},
					],
				},
			},
		},
		{
			name: 'four-items',
			context: {
				data: {
					rows: [
						{
							modifier: 'style-3',
							items: productCategory.slice(0, 4),
						},
					],
				},
			},
		},
		{
			name: 'five-items',
			context: {
				data: {
					rows: [
						{
							modifier: 'style-3',
							items: productCategory.slice(0, 5),
						},
					],
				},
			},
		},
	],
};
