// @siroop/kvasir/source/70-templates/landing-page/landing-page.config.js
const productListData = require('../../60-components/66-organisms/list/list.config');
const banner = require('../../00-data/banner');

module.exports = {
	title: 'Landing Page',
	name: 'landing-page',
	context: {
		productList: {
			items: [
				productListData.context.data.items[0],
				productListData.context.data.items[1],
				productListData.context.data.items[2],
				productListData.context.data.items[3],
			],
			moreUrlLabel: '',
			moreUrl: '',
			modifierListItem: 'vertical',
		},
		productListExtended: {
			items: productListData.context.data.items,
			moreUrlLabel: '',
			moreUrl: '',
			modifierListItem: 'vertical',
		},
		data: {
			banner: {
				showImage: true,
				image: banner[4],
			},
			title: 'Landing Page Template',
			body: `
				<h1>Content Template Subline</h1>
				<h2>Content Template Subline</h2>
				<h3>Content Template Subline</h3>
				<div class="image-left">
					<div class='image'>
						<picture>
							<img src="/img/content/teaser/bg-teaser-home-large-x1.jpg" alt="">
						</picture>
					</div>
					<div class="text">
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores at aut corporis
						cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
					</div>
				</div>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<ul>
				<li>Lorem Ipsum Doler sit amet.</li>
				<li>Consectetur adipisicing elit</li>
				<li><a href="#alink">Amet architecto asperiores</a></li>
				<li>Perspiciatis saepe sequi temporibus vel</li>
				</ul>
				<div class="image-right">
					<div class='image'>
						<picture>
							<img src="/img/content/teaser/bg-teaser-home-large-x1.jpg" alt="">
						</picture>
					</div>
					<div class="text">
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores at aut corporis
						cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
					</div>
				</div>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<p><img src="/img/content/teaser/bg-teaser-home-large-x1.jpg" alt=""></p>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium, amet architecto asperiores,
				at aut corporis cumque dolor doloremque enim iste laborum nam optio perspiciatis saepe sequi temporibus vel, veniam voluptates</p>
				<ol>
				<li>Lorem Ipsum Doler sit amet.</li>
				<li>Consectetur adipisicing elit</li>
				<li><a href="#alink">Amet architecto asperiores</a></li>
				<li>Perspiciatis saepe sequi temporibus vel</li>
				</ol>
			`,
		},
	},
};
