// @siroop/kvasir/source/60-components/64-molecules/input-quantity/input-quantity.js
export const SELECTORS = {
	buttonMinus: '.js-m-input-quantity__button-minus',
	buttonPlus: '.js-m-input-quantity__button-plus',
	inputNumber: '.js-m-input-quantity__input',
	buttonWrap: '.js-m-input-quantity__button-wrap',
};

class InputQuantity {
	constructor($el) {
		this.$el = $el;
		this.buttonMinus = this.$el.find(SELECTORS.buttonMinus);
		this.buttonPlus = this.$el.find(SELECTORS.buttonPlus);
		this.inputNumber = this.$el.find(SELECTORS.inputNumber);
		this.buttonWrap = this.$el.find(SELECTORS.buttonWrap);
		this.minValue = parseInt(this.$el.find(SELECTORS.inputNumber).attr('min'), 10);
		this.maxValue = parseInt(this.$el.find(SELECTORS.inputNumber).attr('max'), 10);

		this.inputNumber.on('input', this.onInputChanges.bind(this));
		this.showButtons();
	}

	onInputChanges() {
		const currentValue = parseInt(this.inputNumber.val(), 10);
		if (currentValue < this.minValue) {
			this.inputNumber.val(this.minValue);
		}
		if (currentValue > this.maxValue) {
			this.inputNumber.val(this.maxValue);
		}
	}

	showButtons() {
		this.buttonWrap.removeClass('is-hidden');
	}
}

export default InputQuantity;
