// @siroop/kvasir/source/00-data/form/billing-address.js
const billingAddress = {
	name: 'billingaddress',
	title: 'Rechnungsadresse',
	fields: {
		gender: {
			title: 'Geschlecht',
			errorMessage: 'Bitte gib etwas ein',
			items: [
				{
					id: 'billing-gender-01',
					name: 'billing-gender',
					value: 'billing-woman',
					label: 'Frau',
					options: {
						required: true,
					},
				},
				{
					id: 'billing-gender-02',
					name: 'billing-gender',
					value: 'billing-man',
					label: 'Herr',
					options: {
						required: true,
					},
				},
			],
		},
		firstName: {
			id: 'billing-first-name',
			name: 'billing-first-name',
			label: 'Vorname',
			autocomplete: 'given-name',
			errorMsg: 'Bitte gib einen Vornamen an.',
			options: {
				type: 'text',
				required: true,
				minlength: 2,
			},
		},
		lastName: {
			id: 'billing-last-name',
			name: 'billing-last-name',
			label: 'Nachname',
			autocomplete: 'family-name',
			errorMsg: 'Bitte gib einen Nachnamen an.',
			options: {
				type: 'text',
				required: true,
				minlength: 2,
			},
		},
		company: {
			id: 'billing-company',
			name: 'billing-company',
			label: 'Firma',
			options: {
				type: 'text',
			},
		},
		street: {
			id: 'billing-street',
			name: 'billing-street',
			label: 'Strasse',
			errorMsg: 'Bitte gib eine Strasse an.',
			options: {
				type: 'text',
				required: true,
				minlength: 2,
			},
		},
		houseNumber: {
			id: 'billing-housenumber',
			name: 'billing-housenumber',
			label: 'Nr.',
			errorMsg: 'Bitte gib eine Hausnummer an.',
			autocomplete: 'off',
			options: {
				type: 'alphanum',
				required: true,
				maxlength: 5,
			},
		},
		extra: {
			id: 'billing-location-extra',
			name: 'billing-location-extra',
			label: 'Adresszusatz',
			options: {
				type: 'text',
			},
		},
		postal: {
			id: 'billing-postal',
			name: 'billing-postal',
			label: 'PLZ',
			errorMsg: 'Bitte gib eine gültige PLZ an.',
			options: {
				type: 'tel',
				required: true,
				maxlength: 4,
				minlength: 4,
				pattern: '[0-9]{4}',
			},
		},
		city: {
			id: 'billing-city',
			name: 'billing-city',
			label: 'Ort',
			errorMsg: 'Bitte gib einen Ort an.',
			options: {
				type: 'text',
				required: true,
				minlength: '2',
			},
		},
	},
};

module.exports = billingAddress;
