// @siroop/kvasir/source/00-data/form/shipping-address.js
const shippingAddress = {
	name: 'shippingaddress',
	title: 'Lieferadresse',
	fields: {
		gender: {
			title: 'Geschlecht',
			errorMessage: 'Bitte gib etwas ein',
			items: [
				{
					id: 'shipping-gender-01',
					name: 'shipping-gender',
					value: 'shipping-woman',
					label: 'Frau',
					options: {
						type: 'radio',
						required: true,
					},
				},
				{
					id: 'shipping-gender-02',
					name: 'shipping-gender',
					value: 'shipping-man',
					label: 'Herr',
					options: {
						type: 'radio',
						required: true,
					},
				},
			],
		},
		firstName: {
			id: 'shipping-first-name',
			name: 'shipping-first-name',
			label: 'Vorname',
			autocomplete: 'given-name',
			errorMsg: 'Bitte gib einen Vornamen an.',
			options: {
				type: 'text',
				minlength: 2,
				required: true,
			},
		},
		lastName: {
			id: 'shipping-last-name',
			name: 'shipping-last-name',
			label: 'Nachname',
			autocomplete: 'family-name',
			errorMsg: 'Bitte gib einen Nachnamen an.',
			options: {
				type: 'text',
				minlength: 2,
				required: true,
			},
		},
		company: {
			id: 'shipping-company',
			name: 'shipping-company',
			label: 'Firma',
			options: {
				type: 'text',
			},
		},
		street: {
			id: 'shipping-street',
			name: 'shipping-street',
			label: 'Strasse',
			errorMsg: 'Bitte gib eine Strasse an.',
			options: {
				type: 'text',
				minlength: 2,
				required: true,
			},
		},
		houseNumber: {
			id: 'shipping-housenumber',
			name: 'shipping-housenumber',
			label: 'Nr.',
			errorMsg: 'Bitte gib eine Hausnummer an.',
			autocomplete: 'off',
			options: {
				type: 'number',
				maxlength: 5,
				required: true,
			},
		},
		extra: {
			id: 'shipping-location-extra',
			name: 'shipping-location-extra',
			label: 'Adresszusatz',
			options: {
				type: 'text',
			},
		},
		postal: {
			id: 'shipping-postal',
			name: 'shipping-postal',
			label: 'PLZ',
			errorMsg: 'Bitte gib eine gültige PLZ an.',
			options: {
				type: 'number',
				minlength: 4,
				maxlength: 4,
				required: true,
			},
		},
		city: {
			id: 'shipping-city',
			name: 'shipping-city',
			label: 'Ort',
			errorMsg: 'Bitte gib einen Ort an.',
			options: {
				type: 'text',
				required: true,
			},
		},
	},
};

module.exports = shippingAddress;
