// @siroop/kvasir/source/70-templates/gift-finder/gift-finder.config.js
const products = require('../../00-data/product/');
const productListData = require('../../60-components/66-organisms/list/list.config');

module.exports = {
	title: 'gift-finder',
	context: {
		data: {
			productList: productListData.context.data,
			url: 'https://giftfinder.siroop.ch/',
			title: 'Einzigartige Geschenke',
			lead: 'Für wen ist es gedacht?',
			modalTitle: 'Alter und Interessen',
			noProducts: {
				title: 'Kein Geschenk?',
				text: 'Such oben jemanden aus, um die besten Geschenke zu finden.',
			},
			navItems: [
				{
					isActive: false,
					text: 'Frau',
					image: 'https://cdn.siroop.ch/cms/2016/11/giftfinder_women-1.png',
				},
				{
					isActive: true,
					text: 'Mann',
					image: 'https://cdn.siroop.ch/cms/2016/11/giftfinder_character_man-1.png',
				},
				{
					isActive: false,
					text: 'Mädchen',
					image: 'https://cdn.siroop.ch/cms/2016/11/giftfinder_girl-1.png',
				},
				{
					isActive: false,
					text: 'Junge',
					image: 'https://cdn.siroop.ch/cms/2016/11/giftfinder_boy-1.png',
				},
			],
			pillItems: [{
				title: 'Alter',
				items: [{
					text: 'Baby',
					modifier: 'toggle a-pill--active',
				}, {
					text: 'Kleinkind',
					modifier: 'toggle',
				}, {
					text: 'Kind',
					modifier: 'toggle',
				},				{
					text: 'Teenager',
					modifier: 'toggle',
				}],
			}, {
				title: 'Interessen',
				items: [{
					text: 'Basteln',
					modifier: 'toggle',
				}, {
					text: 'Bücher',
					modifier: 'toggle a-pill--active',
				}, {
					text: 'Eletronik',
					modifier: 'toggle',
				}, {
					text: 'Ernährung',
					modifier: 'toggle',
				}, {
					text: 'Gaming',
					modifier: 'toggle a-pill--active',
				}, {
					text: 'Kindervelos',
					modifier: 'toggle',
				}, {
					text: 'Kinderzimmer',
					modifier: 'toggle',
				}, {
					text: 'Kochen',
					modifier: 'toggle',
				}, {
					text: 'Mode',
					modifier: 'toggle',
				}, {
					text: 'Musik',
					modifier: 'toggle',
				}, {
					text: 'Outdoor',
					modifier: 'toggle',
				}, {
					text: 'Puzzles',
					modifier: 'toggle a-pill--active',
				}, {
					text: 'Spielzeug',
					modifier: 'toggle',
				}, {
					text: 'Sport',
					modifier: 'toggle',
				}],
			}],
			items: products,
			mobileBackButton: {
				value: 'Geschenke für <strong>Mädchen</strong>',
			},
			modalDisplayButton: {
				value: 'Interessen auswählen (0)',
			},
			filterResetButton: {
				value: 'Filter zurücksetzen',
			},
			applyButton: {
				value: 'Anwenden',
			},
			discardButton: {
				value: 'Abbrechen',
			},
		},
	},
	variants: [
		{
			name: 'is-front',
			context: {
				modifier: 'is-front',
				data: {

				},
			},
		},
		{
			name: 'is-modal-displayed',
			context: {
				modifier: 'is-modal-displayed',
				data: {

				},
			},
		},
	],
};
