// @siroop/kvasir/source/60-components/66-organisms/notification/notification.config.js
module.exports = {
	title: 'Notification',
	name: 'notification',
	context: {
		modifier: '',
		data: {
			text: 'Das Produkt wurde erfolgreich zum Warenkorb hinzugefügt.',
		},
	},
	variants: [
		{
			name: 'with-button',
			context: {
				modifier: 'with-button',
				data: {
					icon: 'checkmark',
					action: {
						url: '#asd',
						text: 'button text',
					},
				},
			},
		},
		{
			name: 'warning',
			context: {
				modifier: 'warning',
				data: {
					icon: 'warning',
				},
			},
		},
		{
			name: 'error',
			context: {
				modifier: 'error',
				data: {
					icon: 'close',
				},
			},
		},
		{
			name: 'information',
			context: {
				modifier: 'information',
				data: {
					icon: 'info',
				},
			},
		},
		{
			name: 'success',
			icon: 'checkmark',
			context: {
				modifier: 'success',
				data: {
					icon: 'checkmark',
				},
			},
		},
		{
			name: 'large',
			context: {
				modifier: 'large',
			},
		},
		{
			name: 'promotion',
			context: {
				modifier: 'promotion js-notification--no-animation',
				data: {
					text: '40% auf vieles für dein Lauf- und Fitnesstraining | Rabatt-Code: TRAINING',
					url: '#',
				},
			},
		},
		{
			name: 'warning-fixed',
			context: {
				modifier: 'warning-fixed js-notification--no-animation',
				data: {
					text: 'Datatrans Maintenance mode from 1AM to 6AM',
					url: '#',
				},
			},
		},
		{
			name: 'bottom right',
			icon: 'checkmark',
			context: {
				modifier: 'success o-notification--bottom-right',
				data: {
					icon: 'checkmark',
				},
			},
		},
	],
};
