// @siroop/kvasir/source/00-data/banner/index.js
const banner = [
	{
		url: '/components/preview/t-category-l3',
		image: {
			mediaQueries: {
				small: {
					query: 'max-width: 500px',
				},
				large: {
					query: 'min-width: 501px',
				},
			},
			images: {
				small: {
					src: '/img/content/teaser/bg-teaser-home-small-x1.jpg',
				},
				large: {
					src: '/img/content/teaser/bg-teaser-home-large-x1.jpg',
				},
			},
			src: '/img/content/teaser/bg-teaser-home-large-x1.jpg',
			alt: '',
		},
		cinemagraph: {
			poster: '/videos/Cinemagraph_Dragqueen.jpg',
			mp4: '/videos/Cinemagraph_Dragqueen.mp4',
			webm: '/videos/Cinemagraph_Dragqueen.webm',
		},
		title: 'Weihnachts&shy;dekoration',
		body: 'Lorem Ipsum doler sit amet, consec tetur adipiscing elit sed.',
		more: {
			text: 'Gibts bei uns',
		},
	},
	{
		url: '/components/preview/t-category-l3',
		cinemagraph: {
			poster: '/videos/cinemagraph.jpg',
			mp4: '/videos/cinemagraph.mp4',
			webm: '/videos/cinemagraph.webm',
		},
		title: 'Weihnachts&shy;dekoration',
		body: 'Lorem Ipsum doler sit amet, consec tetur adipiscing elit sed.',
		more: {
			text: 'Gibts bei uns',
		},
	},
	{
		url: '/components/preview/t-category-l3',
		image: {
			mediaQueries: {
				small: {
					query: 'max-width: 500px',
				},
				large: {
					query: 'min-width: 501px',
				},
			},
			images: {
				small: {
					src: '/img/content/teaser/bg-teaser-home-small-x1.jpg',
				},
				large: {
					src: '/img/content/teaser/bg-teaser-home-large-x1.jpg',
				},
			},
			src: '/img/content/teaser/bg-teaser-home-large-x1.jpg',
			alt: '',
		},
		title: 'Weihnachts&shy;dekoration',
		body: 'Lorem Ipsum doler sit amet, consec tetur adipiscing elit sed.',
		more: {
			text: 'Gibts bei uns',
		},
	},
	{
		url: '/components/preview/t-category-l3',
		image: {
			mediaQueries: {
				small: {
					query: 'max-width: 500px',
				},
				large: {
					query: 'min-width: 501px',
				},
			},
			images: {
				small: {
					src: '/img/content/teaser/bg-teaser-beta-small-x1.jpg',
				},
				large: {
					src: '/img/content/teaser/bg-teaser-beta-large-x1.jpg',
				},
			},
			src: '/img/content/teaser/bg-teaser-home-beta-x1.jpg',
			alt: '',
		},
		title: 'siroop auf beta',
	},
	{
		url: '/components/preview/t-category-l3',
		image: {
			mediaQueries: {
				small: {
					query: 'max-width: 500px',
				},
				large: {
					query: 'min-width: 501px',
				},
			},
			images: {
				small: {
					src: '/img/content/landing-page/banner.jpg',
				},
				large: {
					src: '/img/content/landing-page/banner.jpg',
				},
			},
			src: '/img/content/landing-page/banner.jpg',
			alt: '',
		},
	},
	{
		url: '/components/preview/t-category-l3',
		image: {
			mediaQueries: {
				small: {
					query: 'max-width: 500px',
				},
				large: {
					query: 'min-width: 501px',
				},
			},
			images: {
				small: {
					src: '/img/content/home/home-banner-mobile-jacken.jpg',
				},
				large: {
					src: '/img/content/home/home-banner-desktop-jacken.jpg',
				},
			},
			src: '/img/content/home/home-banner-desktop-jacken.jpg',
			alt: '',
		},
		cinemagraph: {
			poster: '/videos/Cinemagraph_Dragqueen.jpg',
			mp4: '/videos/Cinemagraph_Dragqueen.mp4',
			webm: '/videos/Cinemagraph_Dragqueen.webm',
		},
	},
];

module.exports = banner;
