// @siroop/kvasir/source/70-templates/category-l3/category-l3.config.js
const productsData = require('../../00-data/product/');
const listFooter = require('../../60-components/66-organisms/list-footer/list-footer.config');
const listHeader = require('../../60-components/66-organisms/list-header/list-header.config');

const breadcrumb = [
	{
		url: '/',
		name: 'Sport & Freizeit',
	},
	{
		url: '/',
		name: 'Outdoor',
	},
	{
		url: '/',
		name: 'Wanderrucksack & Taschen',
	},
];

module.exports = {
	title: 'Category L3',
	name: 'category-l3',
	context: {
		data: {
			categoriesLabel: 'Kategorien',
			filtersLabel: 'Filter',
			title: 'Wanderrucksack & Taschen',
			legalText: 'Nahrungsergänzungsmittel sind kein Ersatz für eine abwechslungsreiche und ausgewogene Ernährung sowie eine gesunde Lebensweise. Die angegebene empfohlene tägliche Verzehrmenge bzw. Tagesdosis darf nicht überschritten werden. Ausserhalb der Reichweite von Kindern aufbewahren.',
			showLegalText: true,
			breadcrumb,
			backToSearch: null,
			previousCategoryLink: '/components/preview/t-category-l2?cat=outdoor',
			previousCategoryLabel: 'Outdoor',
			categories: {
				triggerButton: {
					class: 't-category-l3__category-button',
				},
				name: 'Outdoor',
				url: '/components/preview/t-category-l3',
				back: 'Outdoor',
				buttons: [
					{
						name: 'Zurück zu den Produkten',
						modifiers: ['borders'],
						classes: ['js-o-filter-panel__back'],
					},
				],
				items: [
					{
						active: true,
						name: 'Wanderrucksack & Taschen <i>(55)</i>',
						url: '',
					},
					{
						name: 'Camping Zubehör <i>(713)</i>',
						url: '/components/preview/t-category-l2?campingzubehoer',
					},
					{
						name: 'Feldstecher & Teleskop <i>(246)</i>',
						url: '/components/preview/t-category-l2?feldstecherundzubehoer',
					},
					{
						name: 'Fleece & Outdoorhemden <i>(966)</i>',
						url: '/components/preview/t-category-l2?fleeceundoutdoorhemden',
					},
					{
						name: 'Funktionstücher & Schaals <i>(57)</i>',
						url: '/components/preview/t-category-l2?funktionstücherundschaals',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
					{
						name: 'Isomatten <i>(55)</i>',
						url: '/components/preview/t-category-l2?isomatten',
					},
				],
			},
			hasResults: true,
			noResultsMessage: 'Leider finden wir keine Produkte zu deiner Eingabe',
			listFooter: listFooter.context.data,
			listHeader: listHeader.context.data,
			listProducts: {
				items: productsData,
				modifierListItem: 'horizontal',
				moreUrlLabel: '',
				moreUrl: '',
			},
			filters: {
				name: 'Filter',
				back: 'Outdoor',
				buttons: [
					{
						name: 'Abbrechen',
						modifiers: ['borders', 'small'],
						classes: ['js-o-filter-panel__back'],
					},
					{
						name: '156 Ergebnisse anzeigen',
						modifiers: ['filled'],
					},
				],
				triggerButton: {
					class: 't-category-l3__filter-button',
				},
				activeFilters: {
					count: 3,
					resetFilter: 'Alle Filter zurücksetzen',
					items: [
						{
							text: 'Marke: <b>Blackberry</b>',
						},
						{
							text: 'Farbe: <b>Blau</b>',
						},
						{
							text: 'Farbe: <b>Blau</b>',
						},
						{
							text: 'Farbe: <b>Blau</b>',
						},
					],
				},
				items: [
					{
						name: 'Preis',
						value: 'checked',
						back: 'Filter',
						buttons: [
							{
								name: 'Anwenden',
								modifiers: ['filled'],
							},
						],
						active: true,
						content: {
							name: 'Preis',
							component: {
								name: 'o-price-slider',
								data: {
									id: '1',
									currency: 'CHF',
									button: {
										text: 'Anwenden',
									},
									start: {
										min: 5,
										max: 880,
									},
									range: {
										min: 5,
										max: 880,
									},
									label: {
										from: 'Von',
										to: 'Bis',
									},
								},
							},
						},
					},
					{
						name: 'Marke',
						value: 'checked',
						back: 'Filter',
						buttons: [
							{
								name: 'Anwenden',
								modifiers: ['filled'],
							},
						],
						active: true,
						content: {
							name: 'Marke',
							component: {
								name: 'o-checkbox-list',
								data: {
									items: [
										{
											id: 'checkbox-01',
											name: 'checkbxox',
											label: 'Panasonic',
										},
										{
											id: 'checkbox-02',
											name: 'checkbox',
											label: 'Lego',
										},
										{
											id: 'checkbox-03',
											name: 'checkbox',
											label: 'Braun',
										},
										{
											id: 'checkbox-04',
											name: 'checkbox',
											label: 'Samsung',
										},
										{
											id: 'checkbox-05',
											name: 'checkbox',
											label: 'Bosch',
										},
										{
											id: 'checkbox-06',
											name: 'checkbox',
											label: 'Hugo Boss',
										},
										{
											id: 'checkbox-07',
											name: 'checkbox',
											label: 'Apple',
										},
										{
											id: 'checkbox-08',
											name: 'checkbox',
											label: 'LG',
										},
										{
											id: 'checkbox-09',
											name: 'checkbxox',
											label: 'Acer',
										},
										{
											id: 'checkbox-10',
											name: 'checkbox',
											label: 'Asus',
										},
										{
											id: 'checkbox-11',
											name: 'checkbox',
											label: 'Puma',
										},
										{
											id: 'checkbox-12',
											name: 'checkbox',
											label: 'Silvio Tossi',
										},
										{
											id: 'checkbox-13',
											name: 'checkbox',
											label: 'Yamaha',
										},
										{
											id: 'checkbox-14',
											name: 'checkbox',
											label: 'Boss',
										},
										{
											id: 'checkbox-15',
											name: 'checkbox',
											label: 'Chanel',
										},
										{
											id: 'checkbox-16',
											name: 'checkbox',
											label: 'Dior',
										},
									],
									button: 'Anwenden',
								},
							},
						},
					},
					{
						name: 'Grösse',
						value: 'unchecked',
						back: 'Filter',
						buttons: [
							{
								name: 'Anwenden',
								modifiers: ['filled'],
							},
						],
						active: true,
						content: {
							name: 'Grösse',
							component: {
								name: 'o-checkbox-list',
								data: {
									items: [
										{
											id: 'size-checkbox-01',
											name: 'checkbxox',
											label: '34',
										},
										{
											id: 'size-checkbox-02',
											name: 'checkbxox',
											label: '36',
										},
										{
											id: 'size-checkbox-03',
											name: 'checkbox',
											label: '38',
										},
										{
											id: 'size-checkbox-04',
											name: 'checkbox',
											label: '40',
										},
										{
											id: 'size-checkbox-05',
											name: 'checkbox',
											label: '42',
										},
									],
									button: 'Anwenden',
								},
							},
						},
					},
					{
						name: 'Farbe',
						value: 'unchecked',
						back: 'Filter',
						buttons: [
							{
								name: 'Anwenden',
								modifiers: ['filled'],
							},
						],
						active: true,
						content: {
							name: 'Farbe',
							component: {
								name: 'o-checkbox-list',
								data: {
									items: [
										{
											id: 'color-checkbox-01',
											name: 'checkbxox',
											label: 'Schwarz',
										},
										{
											id: 'color-checkbox-02',
											name: 'checkbxox',
											label: 'Braun',
										},
										{
											id: 'color-checkbox-03',
											name: 'checkbox',
											label: 'Grau',
										},
										{
											id: 'color-checkbox-04',
											name: 'checkbox',
											label: 'Gelb',
										},
										{
											id: 'color-checkbox-05',
											name: 'checkbox',
											label: 'Rot',
										},
									],
									button: 'Anwenden',
								},
							},
						},
					},
				],
			},
		},
	},
	variants: [
		{
			name: 'search-scoped',
			context: {
				data: {
					categoriesLabel: 'Kategorien',
					filtersLabel: 'Filter',
					title: 'Wanderrucksack & Taschen',
					breadcrumb,
					backToSearch: {
						text: 'Zurück zu den Suchergebnissen für iPhone',
						url: '#',
					},
					previousCategoryLink: '/components/preview/t-category-l2?cat=outdoor',
					previousCategoryLabel: 'Outdoor',
					categories: {
						triggerButton: {
							class: 't-category-l3__category-button',
						},
						name: 'Outdoor',
						url: '/components/preview/t-category-l3',
						back: 'Outdoor',
						buttons: [
							{
								name: 'Zurück zu den Produkten',
								modifiers: ['borders'],
								classes: ['js-o-filter-panel__back'],
							},
						],
						items: [
							{
								name: 'Camping Zubehör <i>(713)</i>',
								url: '/components/preview/t-category-l2?campingzubehoer',
							},
							{
								name: 'Feldstecher & Teleskop <i>(246)</i>',
								url: '/components/preview/t-category-l2?feldstecherundzubehoer',
							},
							{
								name: 'Fleece & Outdoorhemden <i>(966)</i>',
								url: '/components/preview/t-category-l2?fleeceundoutdoorhemden',
							},
							{
								name: 'Funktionstücher & Schaals <i>(57)</i>',
								url: '/components/preview/t-category-l2?funktionstücherundschaals',
							},
							{
								name: 'Isomatten <i>(55)</i>',
								url: '/components/preview/t-category-l2?isomatten',
							},
							{
								active: true,
								name: 'Wanderrucksack & Taschen <i>(55)</i>',
								url: '',
							},
						],
					},
					hasResults: true,
					noResultsMessage: 'Leider finden wir keine Produkte zu deiner Eingabe',
					listFooter: listFooter.context.data,
					listHeader: listHeader.context.data,
					listProducts: {
						items: productsData,
						modifierListItem: 'horizontal',
						moreUrlLabel: '',
						moreUrl: '',
					},
					filters: {
						name: 'Filter',
						back: 'Outdoor',
						buttons: [
							{
								name: 'Abbrechen',
								modifiers: ['borders', 'small'],
								classes: ['js-o-filter-panel__back'],
							},
							{
								name: '156 Ergebnisse anzeigen',
								modifiers: ['filled'],
							},
						],
						triggerButton: {
							class: 't-category-l3__filter-button',
						},
						activeFilters: {
							resetFilter: 'Alle Filter zurücksetzen',
							items: [
								{
									text: 'Marke: <b>Blackberry</b>',
								},
								{
									text: 'Farbe: <b>Blau</b>',
								},
								{
									text: 'Farbe: <b>Blau</b>',
								},
								{
									text: 'Farbe: <b>Blau</b>',
								},
							],
						},
						items: [
							{
								name: 'Preis',
								back: 'Filter',
								buttons: [
									{
										name: 'Anwenden',
										modifiers: ['filled'],
									},
								],
								active: true,
								content: {
									name: 'Preis',
									component: {
										name: 'o-price-slider',
										data: {
											id: '1',
											currency: 'CHF',
											button: {
												text: 'Anwenden',
											},
											start: {
												min: 5,
												max: 880,
											},
											range: {
												min: 5,
												max: 880,
											},
											label: {
												from: 'Von',
												to: 'Bis',
											},
										},
									},
								},
							},
							{
								name: 'Marke',
								back: 'Filter',
								buttons: [
									{
										name: 'Anwenden',
										modifiers: ['filled'],
									},
								],
								active: true,
								content: {
									name: 'Marke',
									component: {
										name: 'o-checkbox-list',
										data: {
											items: [
												{
													id: 'checkbox-01',
													name: 'checkbxox',
													label: 'Panasonic',
												},
												{
													id: 'checkbox-02',
													name: 'checkbox',
													label: 'Lego',
												},
												{
													id: 'checkbox-03',
													name: 'checkbox',
													label: 'Braun',
												},
												{
													id: 'checkbox-04',
													name: 'checkbox',
													label: 'Samsung',
												},
												{
													id: 'checkbox-05',
													name: 'checkbox',
													label: 'Bosch',
												},
												{
													id: 'checkbox-06',
													name: 'checkbox',
													label: 'Hugo Boss',
												},
												{
													id: 'checkbox-07',
													name: 'checkbox',
													label: 'Apple',
												},
												{
													id: 'checkbox-08',
													name: 'checkbox',
													label: 'LG',
												},
												{
													id: 'checkbox-09',
													name: 'checkbxox',
													label: 'Acer',
												},
												{
													id: 'checkbox-10',
													name: 'checkbox',
													label: 'Asus',
												},
												{
													id: 'checkbox-11',
													name: 'checkbox',
													label: 'Puma',
												},
												{
													id: 'checkbox-12',
													name: 'checkbox',
													label: 'Silvio Tossi',
												},
												{
													id: 'checkbox-13',
													name: 'checkbox',
													label: 'Yamaha',
												},
												{
													id: 'checkbox-14',
													name: 'checkbox',
													label: 'Boss',
												},
												{
													id: 'checkbox-15',
													name: 'checkbox',
													label: 'Chanel',
												},
												{
													id: 'checkbox-16',
													name: 'checkbox',
													label: 'Dior',
												},
											],
											button: 'Anwenden',
										},
									},
								},
							},
						],
					},
				},
			},
		},
	],
};
