// @siroop/kvasir/source/60-components/64-molecules/usp-banner/usp-banner.config.js
module.exports = {
	title: 'USP Banner',
	name: 'usp-banner',
	default: 'small',
	variants: [
		{
			name: 'small',
			context: {
				modifier: 'small',
				data: {
					items: [
						{
							title: 'Gratis Versand ab 50.-',
							icon: {
								name: 'parachute',
								modifiers: ['stroked'],
							},
						},
						{
							title: 'Kostenlose Retoure',
							icon: {
								name: 'boomerang',
								modifiers: ['stroked'],
							},
						},
						{
							title: 'Über 800\'000 Produkte',
							icon: {
								name: 'octopus',
								modifiers: ['stroked'],
							},
						},
						{
							title: 'Top Kundenservice',
							icon: {
								name: 'headset',
								modifiers: ['stroked'],
							},
						},
					],
				},
			},
		},
		{
			name: 'large',
			context: {
				modifier: 'large',
				data: {
					items: [
						{
							title: 'Gratis Versand ab 50.-',
							description: 'Innerhalb der ganzen Schweiz.',
							icon: {
								name: 'parachute-yellow',
								modifiers: ['filled', 'x-large'],
							},
						},
						{
							title: 'Kostenlose Retoure',
							description: 'Sollte es dir doch nicht gefallen.',
							icon: {
								name: 'boomerang-yellow',
								modifiers: ['filled', 'x-large'],
							},
						},
						{
							title: 'Über 800\'000 Produkte',
							description: 'von Schweizer Händlern',
							icon: {
								name: 'octopus-yellow',
								modifiers: ['filled', 'x-large'],
							},
						},
						{
							title: 'Top Kundenservice',
							description: 'Auch per WhatsApp, Facebook und Twitter.',
							icon: {
								name: 'headset-yellow',
								modifiers: ['filled', 'x-large'],
							},
						},
					],
				},
			},
		},
	],
};
