// @siroop/kvasir/source/60-components/64-molecules/usp-banner/usp-banner.js
import ResiroopCarousel from '../../66-organisms/carousel-resiroop/carousel.js';

const SELECTORS = {
	carousel: '.js-m-usp-banner__container',
};

const CLASSES = {
	uspModifierLarge: 'js-m-usp-banner--large',
};

const optionsMainCarousel = el => ({
	selector: el,
	pagination: true,
	loop: true,
	draggable: true,
	duration: 300,
});
class UspBanner {
	constructor($el) {
		this.$el = $el;
		this.el = this.$el.get(0);
		this.window = window;
		this.$window = $(this.window);
		this.carouselElement = this.el.querySelector(SELECTORS.carousel);

		if (this.$el.hasClass(CLASSES.uspModifierLarge)) {
			this.mainCarouselInstance = new ResiroopCarousel(optionsMainCarousel(this.carouselElement));
		}

		this.registerEventListeners();
		this.$window.on('resize', () => this.debounce(this.registerEventListeners(), 100));
	}

	registerEventListeners() {
		// Apply the carousel only on the large USP banner modifier
		if (this.$el.hasClass(CLASSES.uspModifierLarge)) {
			// Apply the carousel only on small viewport
			if (this.window.matchMedia('(min-width: 768px)').matches) {
				if (this.mainCarouselInstance) {
					this.mainCarouselInstance.destroy(true);
				}
			} else {
				this.mainCarouselInstance.init();
			}
		}
	}

	debounce(fn, wait) {
		let timeout;
		return (...args) => {
			clearTimeout(timeout);
			timeout = setTimeout(() => fn.apply(this, args), (wait || 1));
		};
	}
}

export default UspBanner;
