// @siroop/kvasir/karma.conf.js
const webpackConfig = require('./webpack.config.js')();

const sourceGlob = './test/index.js';

module.exports = function karma(config) {
	config.set({
		frameworks: ['jasmine-jquery', 'jasmine', 'viewport'],
		files: [sourceGlob, { pattern: 'fixtures/*.html', watched: true, served: true, included: false }],
		preprocessors: {
			[sourceGlob]: ['webpack'],
		},
		webpack: webpackConfig,
		webpackMiddleware: { noInfo: true },
		reporters: ['spec', 'coverage-istanbul'],
		logLevel: config.LOG_ERROR,
		autoWatch: false,
		browsers: ['PhantomJS'],
		singleRun: true,
		concurrency: Infinity,
		coverageIstanbulReporter: {
			reports: ['text-summary', 'html'],
			dir: './reports/coverage',
			fixWebpackSourcePaths: true,
		},
		viewport: {
			breakpoints: [
				{
					name: 'small',
					size: {
						width: 320,
						height: 480,
					},
				},
				{
					name: 'medium',
					size: {
						width: 768,
						height: 1024,
					},
				},
				{
					name: 'large',
					size: {
						width: 1024,
						height: 900,
					},
				},
			],
		},
	});
};
