// @siroop/kvasir/source/70-templates/checkout-payment/checkout-payment.config.js
module.exports = {
	title: 'Checkout Payment',
	name: 'checkout-payment',
	context: {
		modifier: '',
		data: {
			buttons: {
				next: 'Weiter',
				back: 'Zurück',
			},
			steps: [
				{
					name: 'Persönliche Daten',
					completed: true,
					url: '#1',
				},
				{
					name: 'Zahlungsweise',
					active: true,
				},
				{
					name: 'Bestellübersicht',
				},
			],
			myMethods: {
				title: 'Meine gespeicherte Zahlungsarten',
				items: [
					{
						id: 'radio-01',
						name: 'payment-method',
						value: 'card',
						label: 'Meine Visa',
						svgs: [
							'visa-colored',
						],
						text: '**** **** **** 2345<br /> gültig bis: 05/18',
						options: {
							type: 'radio',
							delete: 'Entfernen',
							checked: 'true',
						},
					},
					{
						id: 'radio-02',
						name: 'payment-method',
						value: 'card',
						label: 'Meine Mastercard',
						svgs: [
							'mastercard-colored',
						],
						text: '**** **** **** 5678<br /> gültig bis: 02/19',
						options: {
							type: 'radio',
							delete: 'Entfernen',
						},
					},
					{
						id: 'radio-03',
						name: 'payment-method',
						value: 'card',
						label: 'Meine Postfinance',
						svgs: [
							'postfinance-colored',
						],
						text: '**** **** **** 6789<br /> gültig bis: 06/20',
						options: {
							type: 'radio',
							delete: 'Entfernen',
						},
					},
				],
			},
			newMethods: {
				title: 'Neue Zahlungsart eintragen',
				items: [
					{
						id: 'radio-04',
						name: 'payment-method',
						value: 'card',
						label: 'Akzeptierte Zahlungsarten',
						svgs: [
							'postfinance-colored',
							'visa-colored',
							'mastercard-colored',
							'amex-colored',
							'paypal-colored',
						],
						options: {
							type: 'radio',
						},
					},
				],

			},

			billingMethod: {
				title: 'Mit Rechnung bezahlen',
				items: [
					{
						id: 'radio-05',
						name: 'payment-method',
						value: 'bill',
						label: 'Rechnung',
						desc: 'Du erhältst von uns per E-Mail eine Rechnung und hast ab Zustellung 30 Tage Zeit, sie zu begleichen.',
						svgs: [
							'bulletin-colored',
						],
						options: {
							type: 'radio',
						},
					},
				],
			},
		},
	},
};
