// @siroop/kvasir/source/60-components/66-organisms/footer/footer.config.js
const socialLinks = require('../../64-molecules/social-links/social-links.config');

module.exports = {
	title: 'Footer',
	name: 'footer',
	context: {
		data: {
			customerService: {
				title: 'Kundenservice',
				items: [
					{
						icon: {
							name: 'faq',
						},
						title: 'FAQ: <a href="/components/preview/t-faq-page">Häufige Fragen</a>',
					},
					{
						icon: {
							name: 'boomerang',
						},
						title: '<a href="http://retoure.siroop.ch">Retoure</a>',
					},
				],
			},
			payment: {
				title: 'Zahlungsarten',
				items: [
					{
						title: 'Visa',
						icon: 'visa-colored',
					},
					{
						title: 'Mastercard',
						icon: 'mastercard-colored',
					},
					{
						title: 'American Express',
						icon: 'amex-colored',
					},
					{
						title: 'Post Finance',
						icon: 'postfinance-colored',
					},
					{
						title: 'Bulletin Covered',
						icon: 'bulletin-colored',
					},
					{
						title: 'Paypal',
						icon: 'paypal-colored',
					},
				],
			},
			security: {
				title: 'Sicher einkaufen',
				items: [
					{
						title: 'Secure Shopping',
						src: '/img/content/swiss-online-secure.png',
					},
				],
			},
			about: {
				title: 'Über Siroop',
				items: [
					{
						url: '#press',
						name: 'Presse',
					},
				],
			},
			work: {
				title: 'Arbeiten bei uns',
				items: [
					{
						url: 'https://jobs.siroop.ch/',
						name: 'siroop Work',
					},
					{
						url: 'https://jobs.siroop.ch/de/offene-stellen/alle/',
						name: 'Alle Offenen Stellen',
					},
				],
			},
			socialLinks: socialLinks.context.data,
			labelSocialLinks: 'Folge uns',
			message: 'Happy Shopping mit Siroop',
			links: [
				{
					url: '#impressum',
					title: 'Impressum',
				},
				{
					url: '#datenschutz',
					title: 'Datenschutz',
				},
				{
					url: '#agb',
					title: 'AGB',
				},
			],
		},
	},
};
