// @siroop/kvasir/source/70-templates/checkout/checkout.config.js
const formBillingAddress = require('../../00-data/form/billing-address');
const formShippingAddress = require('../../00-data/form/shipping-address');
const formPersonalData = require('../../00-data/form/personal-data');
const formContactInfo = require('../../00-data/form/contact-info');
const formCreateAccount = require('../../00-data/form/create-account');

const customData = formPersonalData;
customData.fields.dateofbirth.validation = {
	group: 'billingaddress',
	parsleyTriggerAfterFailure: 'false',
	parsleyRequiredMessage: '',
	parsleyCustomDateMessage: 'Bitte gib ein vollständiges Datum ein',
	parsleyCustomDateCheck: 'true',
	dayMin: '00',
	dayMinMessage: 'Diesen Tag gibt es nicht.',
	dayMax: '31',
	dayMaxMessage: 'Diesen Tag gibt es nicht.',
	monthMax: '12',
	monthMaxMessage: 'Den Monat gibt es nicht.',
	yearMin: '1900',
	yearMinMessage: 'Meister Yoda, bist du es? Du bist zu alt',
	yearMax: '2017',
	yearMaxMessage: 'Prinzessin, liegt dein Alter wirklich in der Zukunft?',
	errorMessage: '',
};

module.exports = {
	title: 'Checkout',
	name: 'checkout',
	context: {
		modifier: '',
		data: {
			formCheckout: {
				labelRequiredFields: '* Pflichtfelder',
				formBillingAddress,
				formCheckShipping: {
					id: 'form-checkout-is-shipping',
					name: 'form-checkout-is-shipping',
					label: 'Abweichende Lieferadresse',
				},
				formShippingAddress,
				formPersonalData: customData,
				formContactInfo,
				formCheckAccount: {
					id: 'form-checkout-is-new-customer',
					name: 'form-checkout-is-new-customer',
					label: 'Ich möchte ein Kundenkonto erstellen',
					benefits: [
						'alle deine bestellungen im Ueberblick',
						'Rechnung downloaden',
					],
				},
				formCreateAccount,
			},
			steps: [
				{
					name: 'Persönliche Daten',
					active: true,
					url: '#1',
				},
				{
					name: 'Zahlungsweise',
				},
				{
					name: 'Bestellübersicht',
				},
			],
			buttonSubmit: 'weiter zur Zahlungsmethode',
		},
	},
};
