// @siroop/kvasir/source/70-templates/customer-address/customer-address.config.js
const customer = require('../../00-data/customer');
const navigationList = require('../../60-components/64-molecules/list-navigation/list-navigation.config');

module.exports = {
	title: 'Customer address',
	name: 'customer-address',
	context: {
		data: {
			title: 'Meine Adressen',
			billingAddress: {
				title: 'Rechnungasadresse',
				items: [
					{
						label: 'Name',
						value: `${customer.billingAddress.prename} ${customer.billingAddress.name}`,
					},
					{
						label: 'Firma',
						value: customer.billingAddress.company,
					},
					{
						label: 'Adresse',
						value: customer.billingAddress.address,
					},
					{
						label: 'Weitere Angaben',
						value: customer.billingAddress.address2,
					},
					{
						label: 'PLZ und Ort',
						value: `${customer.billingAddress.plz} ${customer.billingAddress.city}`,
					},
				],
			},
			shippingAddress: {
				title: 'Lieferadresse',
				items: [
					{
						label: 'Name',
						value: `${customer.shippingAddress.prename} ${customer.shippingAddress.name}`,
					},
					{
						label: 'Firma',
						value: customer.shippingAddress.company,
					},
					{
						label: 'Adresse',
						value: customer.shippingAddress.address,
					},
					{
						label: 'Weitere Angaben',
						value: customer.shippingAddress.address2,
					},
					{
						label: 'PLZ und Ort',
						value: `${customer.shippingAddress.plz} ${customer.shippingAddress.city}`,
					},
				],
			},
			breadcrumb: [
				{
					url: '/',
					name: 'Startseite',
				},
				{
					url: '/components/preview/t-customer',
					name: 'Mein Konto',
				},
				{
					name: 'Meine Adressen',
				},
			],
			navigation: navigationList.context.data,
		},
	},
};
