// @siroop/kvasir/source/00-data/category/single-category.js
const singleCategory = {
	title: 'Das Zürcher Yoga- und Lifestyle- label',
	titleSuffix: '45 Produkte',
	url: '#',
	tag: 'Top Produkt',
	featuredImage: {
		srcset: [
			{
				src: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				width: '50w',
			},
			{
				src: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				width: '100w',
			},
			{
				src: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
				width: '200w',
			},
		],
		src: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg',
		alt: 'Frauenmode',
		srcsetString: 'https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg 50w, https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg 100w, https://cdn.siroop.ch/media/images/sized/b5609204c2f5c3a37cf6a55ed519cf07.400x400.jpg 200w',
		sizes: '',
	},
	backgroundImage: {
		srcset: [
			{
				src: '/img/content/product-families/background-image-1.jpg',
				width: '50w',
			},
			{
				src: '/img/content/product-families/background-image-1.jpg',
				width: '100w',
			},
			{
				src: '/img/content/product-families/background-image-1.jpg',
				width: '200w',
			},
		],
		src: '/img/content/product-families/background-image-1.jpg',
		alt: 'Frauenmode',
		srcsetString: '/img/content/product-families/background-image-1.jpg 50w, /img/content/product-families/background-image-1.jpg 100w, /img/content/product-families/background-image-1.jpg 200w',
		sizes: '',
		color: '#00c5ff',
	},
};

module.exports = singleCategory;
