// @siroop/kvasir/source/60-components/66-organisms/list-accordion/list-accordion.config.js
module.exports = {
	title: 'List Accordion',
	name: 'list-accordion',
	context: {
		data: {
			items: [
				{
					name: 'list-accordion',
					type: 'checkbox',
					id: '1',
					title: 'This is title 1',
					links: [
						{
							title: 'Link one',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link two',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link three',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link four',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link five',
							url: 'http://siroop.ch',
						},
					],
				},
				{
					name: 'list-accordion',
					type: 'checkbox',
					id: '2',
					title: 'This is title 2',
					content: 'Sugar plum gingerbread jujubes topping gummi bears gingerbread chocolate bar icing. Soufflé biscuit tootsie roll carrot cake oat cake marzipan carrot cake. Sweet jujubes jujubes cheesecake cake.',
				},
				{
					name: 'list-accordion',
					type: 'checkbox',
					id: '3',
					title: 'This is a title 3',
					content: 'Oat cake sweet roll biscuit lemon drops toffee tiramisu dessert gingerbread. Dessert caramels cheesecake marzipan chocolate bar halvah halvah lollipop. Sugar plum cake fruitcake dessert wafer bear claw oat cake. Lemon drops toffee cake jelly beans.',
				},
				{
					name: 'list-accordion',
					type: 'checkbox',
					id: '4',
					title: 'This is a title 4',
					links: [
						{
							title: 'Link one',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link two',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link three',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link four',
							url: 'http://siroop.ch',
						},
						{
							title: 'Link five',
							url: 'http://siroop.ch',
						},
					],
				},
				{
					name: 'list-accordion',
					type: 'checkbox',
					id: '5',
					title: 'This is a title 5',
					content: 'Oat cake sweet roll biscuit lemon drops toffee tiramisu dessert gingerbread. Dessert caramels cheesecake marzipan chocolate bar halvah halvah lollipop. Sugar plum cake fruitcake dessert wafer bear claw oat cake. Lemon drops toffee cake jelly beans.',
				},
			],
		},
	},
};
