// @siroop/kvasir/source/60-components/64-molecules/energy-label/energy-label.config.js
module.exports = {
	title: 'Energy Label',
	name: 'energy-label',
	default: 'default',
	variants: [
		{
			name: 'default',
			context: {
				data: {
					text: 'Label',
					sup: '++',
				},
			},
		},
		{
			name: 'a-plus-plus',
			context: {
				modifier: 'lightgreen',
				data: {
					text: 'A',
					sup: '++',
				},
			},
		},
		{
			name: 'a-plus-plus-right-with-desc',
			context: {
				modifier: 'right m-energy-label--lightgreen',
				data: {
					text: 'A',
					sup: '++',
					desc: {
						link: 'Energy efficiency label',
						url: '#',
					},
				},
			},
		},
		{
			name: 'a-plus-plus-right',
			context: {
				modifier: 'right m-energy-label--lightgreen',
				data: {
					text: 'A',
					sup: '++',
				},
			},
		},
		{
			name: 'a-plus',
			context: {
				modifier: 'mediumgreen',
				data: {
					text: 'A',
					sup: '+',
				},
			},
		},
		{
			name: 'a',
			context: {
				modifier: 'darkgreen',
				data: {
					text: 'A',
				},
			},
		},
		{
			name: 'b',
			context: {
				modifier: 'yellow',
				data: {
					text: 'B',
				},
			},
		},
		{
			name: 'c',
			context: {
				modifier: 'lightorange',
				data: {
					text: 'C',
				},
			},
		},
		{
			name: 'd',
			context: {
				modifier: 'darkorange',
				data: {
					text: 'D',
				},
			},
		},
		{
			name: 'e',
			context: {
				modifier: 'red',
				data: {
					text: 'E',
				},
			},
		},
	],
};
