// @siroop/kvasir/source/60-components/64-molecules/button-show-more/index.js
import truncate from 'html-truncate';

const state = {
	expanded: 'is-expanded',
	collapsed: 'is-collapsed',
};

const selectors = {
	excerpt: '[data-excerpt]',
	content: '[data-content]',
	expandButton: '[data-expand]',
	initialChars: 'initial-chars',
	collapseButton: '[data-collapse]',
	suffix: 'suffix',
};

class ButtonShowMore {
	constructor($el) {
		this.$el = $el;
		this.$excerpt = $el.find(selectors.excerpt);
		this.$content = $el.find(selectors.content);
		this.initialChars = this.$excerpt.data(selectors.initialChars);
		this.suffix = this.$excerpt.data(selectors.suffix) ? this.$excerpt.data(selectors.suffix) : '...';
		this.$expandButton = $el.find(selectors.expandButton);
		this.$collapseButton = $el.find(selectors.collapseButton);

		if (this.initialChars) {
			this.$el.addClass(state.collapsed);
			this.clampText(this.$content, this.initialChars, this.suffix);
			this.$expandButton.on('click', this.expand.bind(this));
			this.$collapseButton.on('click', this.collapse.bind(this));
		}
	}

	expand() {
		this.$el.addClass(state.expanded);
	}

	collapse() {
		this.$el.removeClass(state.expanded);
	}

	clampText($el, size, suffix) {
		const clampedText = truncate($el.html(), size, { ellipsis: suffix });
		this.$excerpt.html(clampedText);
	}
}

export default ButtonShowMore;
