// @siroop/kvasir/source/71-online-banners/merchant-deal/merchant-deal.config.js
const merchantDeal = require('../../00-data/online-banner/merchant-deal');

module.exports = {
	title: 'Merchant Deal Banner',
	name: 'merchant-deal',
	default: '300x250',
	context: {
		data: merchantDeal,
	},
	variants: [
		{
			name: '300x250',
			context: {
				modifier: '300x250',
			},
			display: {
				'max-width': '300px',
				'min-width': '300px',
			},
		},
		{
			name: '336x280',
			context: {
				modifier: '336x280',
			},
			display: {
				'max-width': '336px',
				'min-width': '280px',
			},
		},
		{
			name: '320x160',
			context: {
				modifier: '320x160',
			},
			display: {
				'max-width': '320px',
				'min-width': '320px',
			},
		},
		{
			name: '160x600',
			context: {
				modifier: '160x600',
			},
			display: {
				'max-width': '160px',
				'min-width': '160px',
			},
		},
		{
			name: '300x600',
			context: {
				modifier: '300x600',
			},
			display: {
				'max-width': '300px',
				'min-width': '300px',
			},
		},
		{
			name: '728x90',
			context: {
				modifier: '728x90',
			},
			display: {
				'max-width': '728px',
				'min-width': '728px',
			},
		},
		{
			name: '800x250',
			context: {
				modifier: '800x250',
			},
			display: {
				'max-width': '800px',
				'min-width': '800px',
			},
		},
		{
			name: '970x250',
			context: {
				modifier: '970x250',
			},
			display: {
				'max-width': '970px',
				'min-width': '9708px',
			},
		},
		{
			name: '994x250',
			context: {
				modifier: '994x250',
			},
			display: {
				'max-width': '994px',
				'min-width': '994px',
			},
		},
	],
};
