// @siroop/kvasir/source/70-templates/home/home.config.js
const teaser = require('../../00-data/teaser');
const banner = require('../../00-data/banner');
const productsData = require('../../00-data/product');
const categoryData = require('../../60-components/66-organisms/category-list/category-list.config');
const productListData = require('../../60-components/66-organisms/list/list.config');

module.exports = {
	title: 'Home',
	context: {
		data: {
			pageTitle: 'Siroop AG - Dein Schweizer Onlineshop',
			banner: {
				banner: {
					showImage: true,
					image: banner[5],
				},
				carousel: {
					title: 'Abzüglich 50% Rabatt mit Voucher-Code: WINTERSALE ',
					items: productsData.slice(14, 18),
					next: 'Next',
					prev: 'Prev',
				},
			},
			teaserList: {
				rows: [
					{
						modifier: 'style-2',
						items: teaser.slice(0, 6),
					},
				],
			},
			productList: productListData.context.data,
			productsListIntro: 'Top Produkte',
			categoryList: categoryData.context.data,
			categoryListIntro: 'Unser Sortiment',
			beta: {
				title: 'Wir arbeiten mit Hochdruck am neuen Shop.',
				list: '<ul><li>Bestellungen: Du kannst bereits jetzt über den neuen Shop bestellen</li> <li>Kundenkonto: Habe etwas Geduld, bald kannst du dich auch einloggen</li><li>Funktionalitäten: Vielleicht vermisst du einige Funktionalitäten, dann sind diese noch in der Umsetzung.</li></ul>',
				greeting: 'siroop wünscht happy shopping!',
				linkShop: {
					text: 'Du siehst den neuen Beta-Shop.',
					link: {
						name: 'Zurück zum aktuellen Shop',
						url: 'https://siroop.ch',
					},
				},
			},
		},
	},
};
