// @siroop/kvasir/source/70-templates/brand-page-overview/brand-page-overview.config.js
const brandsData = require('../../00-data/brand');
const faker = require('faker/locale/de');

const alphabet = [
	'a', 'b', 'c', 'd', 'e', 'f', 'g',
	'h', 'i', 'j', 'k', 'l', 'm', 'n',
	'o', 'p', 'q', 'r', 's', 't', 'u',
	'v', 'w', 'x', 'y', 'z',
];

const alphabetNav = () => (
	alphabet.map(item => ({
		url: `#${item}`,
		text: item,
	}))
);

const topBrands = () => {
	const out = [];
	const brands = brandsData.map(item => ({
		name: item.name,
		logo: item.logo,
	}));

	for (let i = 0; i < 12; i += 1) {
		out.push(brands[0]);
		out.push(brands[1]);
	}

	return out;
};

const getRandomInt = (min, max) => Math.floor(Math.random() * ((max - min) + 1)) + min;

const getRandomBrand = () => {
	const name = faker.company.companyName();
	const url = '/components/preview/t-brand-page';
	const results = getRandomInt(0, 50000);

	return {
		firstLetter: name.substr(0, 1).toLowerCase(),
		name: `${name} <i>(${results})</i>`,
		url: `${url}?brand=${name}`,
	};
};


const sortByKey = (array, key) => (
	array.sort((a, b) => {
		const x = a[key];
		const y = b[key];

		if (x < y) {
			return -1;
		} else if (x > y) {
			return 1;
		}
		return 0;
	})
);


const allBrands = () => {
	let randomBrands = [];

	// Generate 500 random brands, and sort them alphabetically
	for (let i = 0; i < 500; i += 1) {
		randomBrands.push(getRandomBrand());
	}

	randomBrands = sortByKey(randomBrands, 'name');

	return alphabet.map(letter => ({
		letter,
		items: randomBrands.filter(brand => brand.firstLetter === letter.toLowerCase()),
	}));
};

module.exports = {
	title: 'Brand page Overview',
	context: {
		data: {
			title: 'Marken',
			breadcrumb: {
				items: [
					{
						url: '#',
						name: 'Startseite',
					},
					{
						url: '#',
						name: 'Marken',
					},
				],
			},
			topBrands: {
				title: 'Top Marken',
				items: topBrands(),
			},
			alphabet: {
				title: 'A-Z',
				items: alphabetNav(),
			},
			brands: {
				title: 'Alle Marken',
				items: allBrands(),
			},
		},
	},
};
