// @siroop/kvasir/source/00-data/merchant/index.js
const merchants = [
	{
		image: {
			src: '/img/content/merchants/01-smaller.jpg',
		},
		imageThumbnail: '/img/content/merchants/01-detail-01@2x.png',
		imageShop: '/img/content/merchants/01-detail-01@2x.png',
		imageGallery: [
			{
				thumbnail: '/img/content/merchants/01-detail-01@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-01@2x.png',
				},
				caption: 'Bild Legende',
				url: '//www.google.ch',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-02@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-02@2x.png',
				},
				caption: 'Bild Legende',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-03@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-03@2x.png',
				},
				caption: 'Bild Legende',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-04@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-04@2x.png',
				},
				caption: 'Bild Legende',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-05@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-05@2x.png',
				},
				caption: 'Bild Legende',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-06@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-06@2x.png',
				},
				caption: 'Bild Legende',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-06@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-06@2x.png',
				},
				caption: 'Bild Legende',
			},
			{
				thumbnail: '/img/content/merchants/01-detail-06@2x.png',
				image: {
					src: '/img/content/merchants/01-detail-06@2x.png',
				},
				caption: 'Bild Legende',
			},
		],
		name: 'Fabrikat',
		location: 'Zürich',
		descriptionSummary: 'Papeterie aus Zürich',
		description: 'Papeterieartikel aus Zürich',
		image_right: 'http://placehold.it/200x200',
		url: '#',
		logo: '/img/content/merchants/siroop-logo.png',
	},
	{
		selected: true,
		image: {
			src: '/img/content/merchants/02@2x.png',
		},
		imageThumbnail: '/img/content/merchants/01-detail-01@2x.png',
		imageShop: '/img/content/merchants/01-detail-01@2x.png',
		name: 'Klötzli',
		location: 'Burgdorf',
		descriptionSummary: 'Messer aus Burgdorf',
		description: 'Klötzli aus Burgdorf',
		image_right: 'http://placehold.it/200x200',
		url: '#',
		logo: '/img/content/merchants/brack-logo.png',
	},
	{
		image: {
			src: '/img/content/merchants/03@2x.png',
		},
		imageThumbnail: '/img/content/merchants/01-detail-01@2x.png',
		imageShop: '/img/content/merchants/01-detail-01@2x.png',
		name: 'Fixishop',
		location: 'Zürich',
		descriptionSummary: 'Velo aus Zürich',
		description: 'Fixi Bikes aus Zürich',
		image_right: 'http://placehold.it/200x200',
		url: '#',
		logo: '/img/content/merchants/fust-logo.png',
	},
	{
		image: {
			src: '/img/content/merchants/04@2x.png',
		},
		imageThumbnail: '/img/content/merchants/01-detail-01@2x.png',
		imageShop: '/img/content/merchants/01-detail-01@2x.png',
		name: 'Franz Carl Weber',
		location: 'Zürich',
		descriptionSummary: 'Spielsachen in Zürich',
		description: 'Spielsachen aus Zürich',
		image_right: 'http://placehold.it/200x200',
		url: '#',
		logo: '/img/content/merchants/intersport-logo.png',
	},
	{
		image: {
			src: '/img/content/merchants/05@2x.png',
		},
		imageThumbnail: '/img/content/merchants/01-detail-01@2x.png',
		imageShop: '/img/content/merchants/01-detail-01@2x.png',
		name: 'ASUS',
		location: 'Zürich',
		descriptionSummary: 'Computer aus Zürich',
		description: 'Elektronik aus Zürich',
		image_right: 'http://placehold.it/200x200',
		url: '#',
		logo: '/img/content/merchants/siroop-logo.png',
	},
	{
		image: {
			src: '/img/content/merchants/06@2x.png',
		},
		imageThumbnail: '/img/content/merchants/01-detail-01@2x.png',
		imageShop: '/img/content/merchants/01-detail-01@2x.png',
		name: 'Babyhaus Wehrli',
		location: 'Zürich',
		descriptionSummary: 'Baby Artikel aus Zürich',
		description: 'Babyartikel aus Zürich',
		image_right: 'http://placehold.it/200x200',
		url: '#',
		logo: '/img/content/merchants/siroop-logo.png',
	},
];

module.exports = merchants;
