// @siroop/kvasir/source/00-data/merchant/products.js
const merchantSorting = [
	{
		selected: true,
		name: 'sort price ascending',
		value: 'sort-price-asc',
		icon: 'icon-price-up',
		indexOrder: [1, 2, 0, 3],
	},
	{
		name: 'sort price descending',
		value: 'sort-price-des',
		icon: 'icon-price-down',
		indexOrder: [1, 0, 2, 3],
	},
	{
		name: 'sort name descending',
		value: 'sort-name-asc',
		icon: 'icon-name-up',
		indexOrder: [1, 0, 2, 3],
	},
	{
		name: 'sort name ascending',
		value: 'sort-name-des',
		icon: 'icon-name-down',
		indexOrder: [1, 2, 0, 3],
	},
];

const merchantProducts = [
	{
		name: 'siroop trading AG',
		url: '/components/preview/t-pdp?merchant=siroop',
		id: '1',
		logo: '/img/content/merchants/siroop-logo.png',
		price: {
			priceValue: '18.36',
			priceCurrency: 'CHF',
		},
		selected: true,
		shipping: {
			type: 'standard',
			text: 'Lieferung voraussichtlich am 01.02.2018',
			icon: 'package',
			method: 'Versand',
			price: {
				priceValue: '5.00',
				priceCurrency: 'CHF',
			},

		},
	},
	{
		name: 'Brack AG',
		merchantUnavailable: true,
		text: 'Zurzeit nicht verfügbar',
		url: '/components/preview/t-pdp?merchant=brack',
		id: '2',
		logo: '/img/content/merchants/brack-logo.png',
		price: {
			priceValue: '19.36',
			priceCurrency: 'CHF',
		},
		selected: false,
		shipping: {
			type: 'standard',
			text: 'Lieferung voraussichtlich am 01.02.2018',
			icon: 'package',
			method: 'Versand',
			price: {
				priceValue: '5.00',
				priceCurrency: 'CHF',
			},

		},
	},
	{
		name: 'Fust',
		url: '/components/preview/t-pdp?merchant=fust',
		id: '3',
		logo: '/img/content/merchants/fust-logo.png',
		price: {
			priceValue: '234.36',
			priceCurrency: 'CHF',
		},
		selected: false,
		shipping: {
			type: 'standard',
			text: 'Lieferung voraussichtlich am 01.02.2018',
			icon: 'package',
			method: 'Versand',
			price: {
				priceValue: '5.00',
				priceCurrency: 'CHF',
			},

		},
	},
	{
		name: 'Intersport',
		url: '/components/preview/t-pdp?merchant=intersport',
		id: '4',
		logo: '/img/content/merchants/intersport-logo.png',
		price: {
			priceValue: '19.36',
			priceCurrency: 'CHF',
		},
		selected: false,
		shipping: {
			type: 'standard',
			text: 'Lieferung voraussichtlich am 01.02.2018',
			icon: 'package',
			method: 'Versand',
			price: {
				priceValue: '5.00',
				priceCurrency: 'CHF',
			},

		},
	},
	{
		name: 'Microspot',
		id: '5',
		price: {
			priceValue: '99.90',
			priceCurrency: 'CHF',
		},
		selected: false,
		shipping: {
			type: 'haulier',
			text: 'Lieferung erfolgt per Spedition voraussichtlich bis 01.02.2018',
			tooltip: 'Der Spediteur wird dich telefonisch kontaktieren, um mit dir einen Liefertermin zu vereinbaren',
			icon: 'truck',
			method: 'Versand',
			price: {
				priceValue: '5.00',
				priceCurrency: 'CHF',
			},

		},
	},
];

module.exports = {
	sorting: merchantSorting,
	products: merchantProducts,
};
