// @siroop/kvasir/source/70-templates/checkout-overview/checkout-overview.config.js
const cartProductList = require('../../60-components/66-organisms/cart-product-list/cart-product-list.config');
const cartTotal = require('../../60-components/66-organisms/cart-total/cart-total.config');
const personalData = require('../../00-data/personal');

module.exports = {
	title: 'Checkout Overview',
	name: 'checkout-overview',
	context: {
		modifier: '',
		data: {
			agb: {
				text: 'Mit dieser Bestellung akzeptierst du die <a href="/agb" target="_blank">allgemeinen Geschäftsbedingungen</a> von siroop.',
			},
			age: {
				text: 'Du bestätigst alle Angaben wahrheitsgetreu ausgefüllt zu haben und mind. 18 Jahre alt zu sein.',
			},
			newsletter: {
				text: 'Ja, ich möchte die kostenlose siroop Flaschenpost als Newsletter erhalten, um von aktuellen Angeboten zu profitieren.',
			},

			buttons: {
				next: 'Jetzt bestellen',
				back: 'Zurück',
			},
			headlines: {
				billingAddress: 'Rechnungsadresse',
				deliveryAddress: 'Lieferadresse',
				personalData: 'Persönliche Daten',
				contactInformation: 'Kontaktinformationen',
				paymentMethod: 'Zahlungsweise',
				orderOverview: 'Bestellübersicht',
			},
			products: cartProductList.variants[0].context.data,
			cartTotal: cartTotal.context.data,
			steps: [
				{
					name: 'Persönliche Daten',
					completed: true,
					url: '#1',
				},
				{
					name: 'Zahlungsweise',
					completed: true,
					url: '#2',
				},
				{
					name: 'Bestellübersicht',
					active: true,
				},
			],
			personalData,
		},
	},
};
