// @siroop/kvasir/source/60-components/64-molecules/list-flag/list-flag.config.js
const merchantsData = require('../../../00-data/merchant');

const mappedMerchants = merchantsData.map(merchant => ({
	image: {
		src: merchant.image.src,
	},
	title: merchant.name,
	description: merchant.description,
	url: merchant.url,
}));

module.exports = {
	title: 'List Flag',
	name: 'list-flag',
	context: {
		data: {
			items: [
				{
					icon: {
						name: 'faq',
					},
					title: 'FAQ: <a href="#faq">Häufige Fragen</a>',
				},
				{
					icon: {
						name: 'headset',
					},
					title: 'Gratishotline: <a href="tel:+41800747667">0800 747 667</a>',
					description: 'Mo - Fr 8-19 Uhr, Sa: 10-12 Uhr',
				},
				{
					icon: {
						name: 'whatsapp',
					},
					title: 'Whatsapp: <a href="whatsapp://+41795507702 ">079 550 77 02</a>',
					description: 'nur Messages',
				},
				{
					icon: {
						name: 'mail',
					},
					title: '<a href="mailto:kundenservice@siroop.ch">kundenservice@siroop.ch</a>',
				},
			],
		},
	},
	variants: [
		{
			name: 'yellow',
			context: {
				modifier: 'yellow',
				data: {
					items: [
						{
							icon: {
								name: 'parachute',
							},
							title: 'Gratis Versand',
							description: 'Innerhalb der ganzen Schweiz.',
						},
						{
							icon: {
								name: 'boomerang',
							},
							title: 'Kostenlose Retoure',
							description: 'Sollte es dir doch nicht gefallen.',
						},
						{
							icon: {
								name: 'octopus',
							},
							title: 'Grosses Sortiment, riesige Auswahl',
							description: 'Hunderttausende Produkte für dich.',
						},
						{
							icon: {
								name: 'pickup',
							},
							title: 'Pick-up Stationen in deiner Nähe',
							description: 'Und noch weitere Liefermöglichkeiten.',
						},
					],
				},
			},
		},
		{
			name: 'image',
			context: {
				modifier: 'image',
				data: {
					items: mappedMerchants.slice(0, 4),
				},
			},
		},
	],
};
