// @siroop/kvasir/source/60-components/64-molecules/voucher-box/voucher-box.js
const selectors = {
	deadline: '.js-m-voucher-box__deadline',
	deadlineTime: '.js-m-voucher-box__deadline-time',
	labelExpired: '.js-m-voucher-box__label-expired',
	dataCounterTime: 'data-counter-time',
};

class VoucherBox {
	constructor($el) {
		this.el = $el[0];
		this.deadline = this.el.querySelector(selectors.deadline);
		this.deadlineTime = this.el.querySelector(selectors.deadlineTime);
		this.labelExpired = this.el.querySelector(selectors.labelExpired);
		this.dataCounterTime = this.deadline.getAttribute(selectors.dataCounterTime);
		this.currentSecs = this.dataCounterTime;
		window.setTimeout(() => this.timeDecrement(), 1000);
	}

	timeDecrement() {
		this.displayedSecs = this.currentSecs % 60;
		this.displayedMin = Math.floor(this.currentSecs / 60) % 60;
		this.displayedHrs = Math.floor(this.currentSecs / 60 / 60);

		if (this.displayedMin <= 9) this.displayedMin = `0${this.displayedMin}`;
		if (this.displayedSecs <= 9) this.displayedSecs = `0${this.displayedSecs}`;

		this.currentSecs -= 1;

		if (this.displayedHrs < 24) {
			this.deadline.classList.add('visible');
			this.deadlineTime.innerHTML = `${this.displayedHrs}:${this.displayedMin}:${this.displayedSecs}`;
		}

		if (this.currentSecs === -1) {
			this.labelExpired.classList.add('visible');
			this.deadline.classList.remove('visible');
		} else {
			setTimeout(() => this.timeDecrement(), 1000);
		}
	}
}

export default VoucherBox;
