// @siroop/kvasir/source/60-components/66-organisms/header/header.config.js
const navigation = require('../../66-organisms/navigation-primary/navigation-primary.config');
const uspBanner = require('../../64-molecules/usp-banner/usp-banner.config');
const searchbar = require('../../64-molecules/searchbar/searchbar.config');
const logo = require('../../64-molecules/logo/logo.config');
const languageSwitcher = require('../../64-molecules/language-switcher/language-switcher.config');

module.exports = {
	title: 'Header',
	name: 'header',
	context: {
		data: {
			menu: 'Menu',
			myAccount: {
				label: 'Anmelden',
				items: [
					{
						label: 'Meine Benutzerdaten',
						url: '/components/preview/t-customer',
					},
					{
						label: 'Meine Adressen',
						url: '/components/preview/t-customer-address',
					},
					{
						label: 'Meine Zahlungsgmethoden',
						url: '/components/preview/t-customer-payment',
					},
					{
						label: 'Logout',
						url: '#logout',
					},
				],
			},
			wishlist: {
				text: 'Wunschzettel',
				numberOfItems: 99,
			},
			cart: {
				text: 'Warenkorb',
				numberOfItems: 99,
				desc: 'Zum Warenkorb. Sie haben 5 Artikel im Warenkorb',
			},
			loggedIn: false,
			uspBanner: uspBanner.variants[0].context.data,
			navigation: navigation.context.data,
			searchbar: searchbar.context.data,
			logo: logo.context.data,
			languageSwitcher: languageSwitcher.context.data,
			shortlinks: {
				items: [
					{
						label: 'Hilfe',
						url: '#',
					},
					{
						label: 'Retouren',
						url: '#',
					},
				],
			},
		},
	},
};
