// @siroop/kvasir/source/05-materials/05-colors/colors.config.js
module.exports = {
	title: 'Colors',
	status: 'live',
	tags: ['resiroop', 'loki'],
	default: 'primary_colors',
	context: {
		hues: [5, 10, 20, 30, 40, 50, 60, 70, 80, 90],
	},
	variants: [
		{
			label: 'Brand',
			name: 'brand_colors',
			context: {
				modifier: 'colors-brand',
				colors: [
					'teal',
					'red',
					'yellow',
				],
			},
		},
		{
			label: 'Primary',
			name: 'primary_colors',
			context: {
				modifier: 'colors-primary',
				colors: [
					'orange',
					'green',
					'grey',
				],
			},
		},
		{
			label: 'Secondary',
			name: 'secondary_colors',
			context: {
				modifier: 'colors-secondary',
				colors: [
					'whatsapp',
					'facebook',
					'twitter',
					'instagram',
					'linkedin',
				],
			},
		},
		{
			name: 'hues_grey',
			label: 'Grey',
			context: {
				modifier: 'hue-grey',
				isHue: true,
				colors: ['grey'],
			},
		},
		{
			name: 'hues_red',
			label: 'Red',
			context: {
				modifier: 'hue-red',
				isHue: true,
				colors: ['red'],
			},
		},
		{
			name: 'hues_orange',
			label: 'Orange',
			context: {
				modifier: 'hue-orange',
				isHue: true,
				colors: ['orange'],
			},
		},
		{
			name: 'hues_yellow',
			label: 'Yellow',
			context: {
				modifier: 'hue-yellow',
				isHue: true,
				colors: ['yellow'],
			},
		},
		{
			name: 'hues_teal',
			label: 'Teal',
			context: {
				modifier: 'hue-teal',
				isHue: true,
				colors: ['teal'],
			},
		},
		{
			name: 'hues_green',
			label: 'Green',
			context: {
				modifier: 'hue-green',
				isHue: true,
				colors: ['green'],
			},
		},
	],
};
