// @siroop/kvasir/source/60-components/64-molecules/button-toggle-element/index.js
const state = {
	toggled: 'is-toggled',
};

const selectors = {
	toggleButton: '[data-toggle]',
};

const attributes = {
	toggleTarget: 'data-target-to-toggle',
};

class ButtonToggleElement {
	constructor($el) {
		this.el = $el.get(0); // preparation for easy removal of query later
		this.toggleButton = this.el.querySelector(selectors.toggleButton);

		if (this.toggleButton) {
			this.targetToToggleAttribute = this.toggleButton.getAttribute(attributes.toggleTarget);
			this.toggleButton.classList.add(state.toggled);
			this.targetsToToggle = document.querySelectorAll(this.targetToToggleAttribute);

			this.toggleTargets(this.targetsToToggle);

			this.toggleButton.addEventListener('click', ((evt) => {
				evt.target.classList.remove(state.toggled);
				this.toggleTargets(this.targetsToToggle);
			}));
		}
	}

	toggleTargets(targetsToToggle) { // eslint-disable-line class-methods-use-this
		[].forEach.call(targetsToToggle, (element) => {
			element.classList.toggle(state.toggled);
		});
	}
}

export default ButtonToggleElement;
