// @siroop/kvasir/source/60-components/64-molecules/banner-flexbox/banner-flexbox.config.js
const banner = require('../../../00-data/banner');

module.exports = {
	title: 'Banner Flexbox',
	name: 'banner-flexbox',
	default: 'default',
	variants: [
		{
			name: 'default',
			context: {
				modifier: 'image',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'image',
			context: {
				modifier: 'image',
				data: {
					showImage: true,
					image: banner[2],
				},
			},
		},
		{
			name: 'cinemagraph',
			context: {
				modifier: 'cinemagraph',
				data: {
					showCinemagraph: true,
					image: banner[1],
				},
			},
		},
		{
			name: 'position-style-1',
			context: {
				modifier: 'position-style-1',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'position-style-2',
			context: {
				modifier: 'position-style-2',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'position-style-3',
			context: {
				modifier: 'position-style-3',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'position-style-4',
			context: {
				modifier: 'position-style-4',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'color-style-1',
			context: {
				modifier: 'color-style-1',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'color-style-2',
			context: {
				modifier: 'color-style-2',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'color-style-3',
			context: {
				modifier: 'color-style-3',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'color-style-4',
			context: {
				modifier: 'color-style-4',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'color-style-5',
			context: {
				modifier: 'color-style-5',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
		{
			name: 'color-style-6',
			context: {
				modifier: 'color-style-6',
				data: {
					showImage: true,
					image: banner[0],
				},
			},
		},
	],
};
