// @siroop/kvasir/source/60-components/64-molecules/banner/banner.config.js
const banner = require('../../../00-data/banner');

module.exports = {
	title: 'Banner',
	name: 'banner',
	default: 'default',
	variants: [
		{
			name: 'default',
			context: {
				data: banner[0],
			},
		},
		{
			name: 'position-style-1',
			context: {
				modifier: 'position-style-1',
				data: banner[0],
			},
		},
		{
			name: 'position-style-2',
			context: {
				modifier: 'position-style-2',
				data: banner[0],
			},
		},
		{
			name: 'position-style-3',
			context: {
				modifier: 'position-style-3',
				data: banner[0],
			},
		},
		{
			name: 'position-style-4',
			context: {
				modifier: 'position-style-4',
				data: banner[0],
			},
		},
		{
			name: 'color-style-1',
			context: {
				modifier: 'color-style-1',
				data: banner[0],
			},
		},
		{
			name: 'color-style-2',
			context: {
				modifier: 'color-style-2',
				data: banner[0],
			},
		},
		{
			name: 'color-style-3',
			context: {
				modifier: 'color-style-3',
				data: banner[0],
			},
		},
		{
			name: 'color-style-4',
			context: {
				modifier: 'color-style-4',
				data: banner[0],
			},
		},
		{
			name: 'color-style-5',
			context: {
				modifier: 'color-style-5',
				data: banner[0],
			},
		},
		{
			name: 'color-style-6',
			context: {
				modifier: 'color-style-6',
				data: banner[0],
			},
		},
	],
};
