// @siroop/kvasir/source/70-templates/pdp/pdp.js
const classes = {
	elementsShake: 't-pdp__variants--shake',
	elementsFocus: 't-pdp__variants--focus',
	buttonDisabled: 't-pdp__call-to-action-button--disabled',
};

const selectors = {
	ctaButton: '.js-t-pdp__call-to-action-button',
};

class Pdp {
	constructor() {
		this.ctaButtons = document.querySelectorAll(selectors.ctaButton);
		for (let i = 0; i < this.ctaButtons.length; i += 1) {
			this.ctaButton = this.ctaButtons[i];
			this.shakeSelector = this.ctaButton.getAttribute('data-shake-elements') || '';

			if (this.shakeSelector.length > 0) {
				this.elementsToShake = document.querySelector(this.shakeSelector);
				if (this.elementsToShake) {
					this.ctaButton.addEventListener('click', () => this.onButtonCtaClick());
					this.elementsToShake.addEventListener('animationend', () => this.onAnimationEnd());
				}
			}
		}
	}

	onButtonCtaClick() {
		this.elementsToShake.classList.add(classes.elementsShake, classes.elementsFocus);
	}

	onAnimationEnd() {
		this.elementsToShake.classList.remove(classes.elementsShake);
	}
}

export default Pdp;
