// @siroop/kvasir/source/70-templates/pdp/pdp.config.js
const singleProduct = require('../../00-data/product/');
const merchants = require('../../00-data/merchant/products');
const productListData = require('../../60-components/66-organisms/list/list.config');

const productImages = singleProduct[0].images;
const manufacturer = singleProduct[0].manufacturer;
const mainAttributes = singleProduct[0].mainAttributes;
const specifications = singleProduct[0].specifications;
const firstMerchantProduct = merchants.products[0];
const firstMerchantName = firstMerchantProduct.name;
const description = singleProduct[0].description;

const notification = {
	text: 'Das Produkt wurde erfolgreich zum Warenkorb hinzugefügt.',
	icon: 'checkmark',
	action: {
		text: 'Zum Warenkorb',
		url: '/components/preview/t-cart',
	},
};

const breadcrumb = [
	{
		url: '/components/detail/t-category-l1',
		name: 'Computer & Elektronik',
	},
	{
		url: '/components/detail/t-category-l2',
		name: 'Smartphone & Handy',
	},
	{
		url: '/components/preview/t-category-l3--default',
		name: 'Smartphone',
	},
	{
		url: '/',
		name: 'Apple iPhone 7',
	},
];

const backSearch = {
	url: '/components/preview/t-search-results',
	backText: 'Zurück zu Suchergebnisse für',
	searchTerm: 'iPhone',
};

const groups = {
	title: singleProduct[0].groups.name,
	items: singleProduct[0].groups.items,
};

const variants = {
	title: singleProduct[0].variants.name,
	labelChooseVariant: 'wählen',
	selectedVariantName: null,
	items: [...singleProduct[0].variants.items, ...singleProduct[0].variants.items],
};

const merchantInfo = {
	labelInStock: 'Auf Lager',
	labelOutOfStock: 'Nicht auf Lager',
	shippingDate: 'vsl. Liefertermin am 31.01.2017',
	shippingCost: {
		pricePrefix: 'Versandkosten: ',
		priceCurrency: firstMerchantProduct.shipping.price.priceCurrency,
		priceValue: firstMerchantProduct.shipping.price.priceValue,
		priceSuffix: '',
	},
	moreLabel: `${merchants.products.length} weitere Angebote`,
	merchantTooltip: 'CHE-100.273.774&#xa;1Advd.ch / Forbidden Planet AG&#xa;Althardstrasse 150&#xa;8105 Regensdorf',
};

const mappedImages = productImages.map(item => ({
	caption: item.title,
	image: {
		src: item.src,
		srcset: item.srcset,
		alt: item.alt,
		srcsetString: item.srcsetString,
	},
}));

module.exports = {
	title: 'PDP',
	context: {
		data: {
			wishlist: {
				text: 'Auf den Wunschzettel',
			},
			notification,
			breadcrumb,
			backSearch,
			labelCartButton: 'In den Warenkorb',
			labelOutOfStock: 'Dieses Produkt ist leider nicht verfügbar',
			labelShippingCost: 'Versandkosten',
			labelMainAttributes: 'Hauptattribute',
			labelSpecifications: 'Spezifikationen',
			labelMerchantInformationTitle: 'Händlerinformationen',
			title: singleProduct[0].title,
			description,
			carousel: {
				labelNext: 'Next',
				labelPrev: 'Prev',
				items: mappedImages,
			},
			groups,
			variants,
			manufacturer,
			mainAttributes,
			specifications,
			price: singleProduct[0].price,
			modalWindow: {
				title: 'Energy efficency details',
				close: 'Close this dialog window',
				identifier: 't-pdp__energy-label',
				image: {
					srcset: [
						{
							src: '//placehold.it/50x50',
							width: '50w',
						},
						{
							src: '//placehold.it/100x100',
							width: '100w',
						},
						{
							src: '//placehold.it/200x200',
							width: '200w',
						},
						{
							src: '//placehold.it/500x500',
							width: '500w',
						},
						{
							src: '//placehold.it/1000x1000',
							width: '1000w',
						},
					],
					srcsetString: '//placehold.it/50x50 50w, //placehold.it/100x100 100w, //placehold.it/200x200 200w, //placehold.it/500x500 500w, //placehold.it/1000x1000 1000w',
					src: '//placehold.it/200x200',
					alt: 'A responsive image',
					sizes: '',
				},
			},
			merchantInfo,
			priceShipping: merchantInfo.shippingCost,
			merchantName: firstMerchantName,
			merchants: {
				title: 'Angebot von',
				products: merchants.products,
				sorting: merchants.sorting,
			},
			labelReadmore: 'mehr lesen',
			labelManufacturer: 'Hersteller',
			recommendationsText: 'Kunden, die dieses Produkt angesehen haben, interessieren sich auch für:',
			productList: productListData.context.data,
		},
	},
};
